<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'><head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"></meta>
<title>arrayQualityMetrics</title>

<script language="JavaScript" src="arrayQualityMetrics.js"></script>
<link rel="stylesheet" type="text/css" href="arrayQualityMetrics.css"></link>
</head>
<body onload="reportinit()"><hr><h1>arrayQualityMetrics</h1><hr><UL><a href="#S1"><br><li class='tocsection'>Section 1: Between array comparison</li><UL></a><li class='tocmodule'>Distances between arrays</li><li class='tocmodule'>Principal Component Analysis</li></UL><a href="#S2"><br><li class='tocsection'>Section 2: Array intensity distributions</li><UL></a><li class='tocmodule'>Boxplots</li><li class='tocmodule'>Density plots</li></UL><a href="#S3"><br><li class='tocsection'>Section 3: Variance mean dependence</li><UL></a><li class='tocmodule'>Standard deviation versus rank of the mean</li></UL><a href="#S4"><br><li class='tocsection'>Section 4: Individual array quality</li><UL></a><li class='tocmodule'>MA plots</li></UL></UL><hr><a name="arraymetadata" id="arraymetadata-h" href="javascript: toggle('arraymetadata')" style="text-decoration:none;font-weight:bold;font-size:larger">- Array metadata and outlier detection overview</a><br>

<div id="arraymetadata-b" style="display:block">
<table style="margin-left:auto;text-align:right;" border="1">
<tr>
<td style="font-weight:bold"> </td><td style="font-weight:bold">array</td><td style="font-weight:bold">sampleNames</td><td style="font-weight:bold"><a href="#hm">*1</a></td><td style="font-weight:bold"><a href="#box">*2</a></td><td style="font-weight:bold"><a href="#ma">*3</a></td><td style="font-weight:bold">title</td><td style="font-weight:bold">geo_accession</td><td style="font-weight:bold">status</td><td style="font-weight:bold">submission_date</td><td style="font-weight:bold">last_update_date</td><td style="font-weight:bold">type</td><td style="font-weight:bold">channel_count</td><td style="font-weight:bold">source_name_ch1</td><td style="font-weight:bold">organism_ch1</td><td style="font-weight:bold">characteristics_ch1</td><td style="font-weight:bold">characteristics_ch1.1</td><td style="font-weight:bold">characteristics_ch1.2</td><td style="font-weight:bold">treatment_protocol_ch1</td><td style="font-weight:bold">growth_protocol_ch1</td><td style="font-weight:bold">molecule_ch1</td><td style="font-weight:bold">extract_protocol_ch1</td><td style="font-weight:bold">label_ch1</td><td style="font-weight:bold">label_protocol_ch1</td><td style="font-weight:bold">taxid_ch1</td><td style="font-weight:bold">hyb_protocol</td><td style="font-weight:bold">scan_protocol</td><td style="font-weight:bold">description</td><td style="font-weight:bold">data_processing</td><td style="font-weight:bold">platform_id</td><td style="font-weight:bold">contact_name</td><td style="font-weight:bold">contact_email</td><td style="font-weight:bold">contact_laboratory</td><td style="font-weight:bold">contact_department</td><td style="font-weight:bold">contact_institute</td><td style="font-weight:bold">contact_address</td><td style="font-weight:bold">contact_city</td><td style="font-weight:bold">contact_state</td><td style="font-weight:bold">contact_zip/postal_code</td><td style="font-weight:bold">contact_country</td><td style="font-weight:bold">supplementary_file</td><td style="font-weight:bold">data_row_count</td><td style="font-weight:bold">cell line:ch1</td><td style="font-weight:bold">tissue type:ch1</td><td style="font-weight:bold">treatment:ch1</td></tr>
<tr>
<td bgcolor="#ffffff"><input type='checkbox' name='ReportObjectCheckBoxes' value='' onchange='checkboxEvent(1)'></td><td bgcolor="#ffffff">1</td><td bgcolor="#ffffff">GSM2589734</td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff">MDA231-LM2_DMSO_Vehicle, biological rep1</td><td bgcolor="#ffffff">GSM2589734</td><td bgcolor="#ffffff">Public on Aug 26 2018</td><td bgcolor="#ffffff">Apr 26 2017</td><td bgcolor="#ffffff">Aug 27 2018</td><td bgcolor="#ffffff">RNA</td><td bgcolor="#ffffff">1</td><td bgcolor="#ffffff">MDA231-LM2 breast cancer cells, treated with DMSO vehicle, 48h</td><td bgcolor="#ffffff">Homo sapiens</td><td bgcolor="#ffffff">cell line: MDA231-LM2</td><td bgcolor="#ffffff">tissue type: breast cancer</td><td bgcolor="#ffffff">treatment: DMSO_Vehicle</td><td bgcolor="#ffffff">MDA231-LM2 cells were seeded at a density of approximately 28.000 cells/cm2 and let to attach to the culture dish overnight. Then cells were treated with either a vehicle control (DMSO) or 5nM paclitaxel for 48h prior to lysis for RNA extraction.</td><td bgcolor="#ffffff">MDA231-LM2 cells were grown in DMEM GlutaMAX supplemented with 10% vol/vol fetal bovine serum (FBS), 100 IU/ml penicillin, 100 μg/ml streptomycin and 1 μg/ml amphotericin B.</td><td bgcolor="#ffffff">total RNA</td><td bgcolor="#ffffff">RNA extraction was performed using the RNEasy Kit (Qiagen) according to manufacturer's instructions</td><td bgcolor="#ffffff">biotin</td><td bgcolor="#ffffff">The Affymetrix Gene Chip 3´IVT PLUS Reagent Kit was used to generate labeled single-stranded cRNA from input amounts of 100ng total RNA. The label used was biotin.</td><td bgcolor="#ffffff">9606</td><td bgcolor="#ffffff">11 ug of fragmented and labeled cRNA were hybridized for 17 hours at 45°C on Affymetrix GeneChip® Human Genome U133 Plus 2.0 Arrays, according to user manual GeneChip® 3' IVT PLUS Reagent Kit, P/N 703210.</td><td bgcolor="#ffffff">Gene Expression Microarrays were scanned using the Affymetrix GeneChip® Scanner 3000 according to GeneChip® Expression Wash, Stain and Scan Manual for Cartridge Arrays  (P/N 702731)</td><td bgcolor="#ffffff">Gene expression of MDA231-LM2 cells cultured as a monolayer</td><td bgcolor="#ffffff">The raw data was processed using R and bioconductor package affy; normalization method is quantile normalization.</td><td bgcolor="#ffffff">GPL570</td><td bgcolor="#ffffff">Jacob,,Insua-Rodríguez</td><td bgcolor="#ffffff">j.insuarodriguez dkfz.de</td><td bgcolor="#ffffff">Oskarsson Lab</td><td bgcolor="#ffffff">Division of Stem Cells and Cancer</td><td bgcolor="#ffffff">German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ)</td><td bgcolor="#ffffff">Im Neuenheimer Feld 280</td><td bgcolor="#ffffff">Heidelberg</td><td bgcolor="#ffffff">Baden-Württemberg</td><td bgcolor="#ffffff">69120</td><td bgcolor="#ffffff">Germany</td><td bgcolor="#ffffff">ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2589nnn/GSM2589734/suppl/GSM2589734_DMSO_1.CEL.gz</td><td bgcolor="#ffffff">54675</td><td bgcolor="#ffffff">MDA231-LM2</td><td bgcolor="#ffffff">breast cancer</td><td bgcolor="#ffffff">DMSO_Vehicle</td></tr>
<tr>
<td bgcolor="#d0d0ff"><input type='checkbox' name='ReportObjectCheckBoxes' value='' onchange='checkboxEvent(2)'></td><td bgcolor="#e0e0f0">2</td><td bgcolor="#d0d0ff">GSM2589735</td><td bgcolor="#e0e0f0"></td><td bgcolor="#d0d0ff"></td><td bgcolor="#e0e0f0"></td><td bgcolor="#d0d0ff">MDA231-LM2_DMSO_Vehicle, biological rep2</td><td bgcolor="#e0e0f0">GSM2589735</td><td bgcolor="#d0d0ff">Public on Aug 26 2018</td><td bgcolor="#e0e0f0">Apr 26 2017</td><td bgcolor="#d0d0ff">Aug 27 2018</td><td bgcolor="#e0e0f0">RNA</td><td bgcolor="#d0d0ff">1</td><td bgcolor="#e0e0f0">MDA231-LM2 breast cancer cells, treated with DMSO vehicle, 48h</td><td bgcolor="#d0d0ff">Homo sapiens</td><td bgcolor="#e0e0f0">cell line: MDA231-LM2</td><td bgcolor="#d0d0ff">tissue type: breast cancer</td><td bgcolor="#e0e0f0">treatment: DMSO_Vehicle</td><td bgcolor="#d0d0ff">MDA231-LM2 cells were seeded at a density of approximately 28.000 cells/cm2 and let to attach to the culture dish overnight. Then cells were treated with either a vehicle control (DMSO) or 5nM paclitaxel for 48h prior to lysis for RNA extraction.</td><td bgcolor="#e0e0f0">MDA231-LM2 cells were grown in DMEM GlutaMAX supplemented with 10% vol/vol fetal bovine serum (FBS), 100 IU/ml penicillin, 100 μg/ml streptomycin and 1 μg/ml amphotericin B.</td><td bgcolor="#d0d0ff">total RNA</td><td bgcolor="#e0e0f0">RNA extraction was performed using the RNEasy Kit (Qiagen) according to manufacturer's instructions</td><td bgcolor="#d0d0ff">biotin</td><td bgcolor="#e0e0f0">The Affymetrix Gene Chip 3´IVT PLUS Reagent Kit was used to generate labeled single-stranded cRNA from input amounts of 100ng total RNA. The label used was biotin.</td><td bgcolor="#d0d0ff">9606</td><td bgcolor="#e0e0f0">11 ug of fragmented and labeled cRNA were hybridized for 17 hours at 45°C on Affymetrix GeneChip® Human Genome U133 Plus 2.0 Arrays, according to user manual GeneChip® 3' IVT PLUS Reagent Kit, P/N 703210.</td><td bgcolor="#d0d0ff">Gene Expression Microarrays were scanned using the Affymetrix GeneChip® Scanner 3000 according to GeneChip® Expression Wash, Stain and Scan Manual for Cartridge Arrays  (P/N 702731)</td><td bgcolor="#e0e0f0">Gene expression of MDA231-LM2 cells cultured as a monolayer</td><td bgcolor="#d0d0ff">The raw data was processed using R and bioconductor package affy; normalization method is quantile normalization.</td><td bgcolor="#e0e0f0">GPL570</td><td bgcolor="#d0d0ff">Jacob,,Insua-Rodríguez</td><td bgcolor="#e0e0f0">j.insuarodriguez dkfz.de</td><td bgcolor="#d0d0ff">Oskarsson Lab</td><td bgcolor="#e0e0f0">Division of Stem Cells and Cancer</td><td bgcolor="#d0d0ff">German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ)</td><td bgcolor="#e0e0f0">Im Neuenheimer Feld 280</td><td bgcolor="#d0d0ff">Heidelberg</td><td bgcolor="#e0e0f0">Baden-Württemberg</td><td bgcolor="#d0d0ff">69120</td><td bgcolor="#e0e0f0">Germany</td><td bgcolor="#d0d0ff">ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2589nnn/GSM2589735/suppl/GSM2589735_DMSO_2.CEL.gz</td><td bgcolor="#e0e0f0">54675</td><td bgcolor="#d0d0ff">MDA231-LM2</td><td bgcolor="#e0e0f0">breast cancer</td><td bgcolor="#d0d0ff">DMSO_Vehicle</td></tr>
<tr>
<td bgcolor="#ffffff"><input type='checkbox' name='ReportObjectCheckBoxes' value='' onchange='checkboxEvent(3)'></td><td bgcolor="#ffffff">3</td><td bgcolor="#ffffff">GSM2589736</td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff">MDA231-LM2_DMSO_Vehicle, biological rep3</td><td bgcolor="#ffffff">GSM2589736</td><td bgcolor="#ffffff">Public on Aug 26 2018</td><td bgcolor="#ffffff">Apr 26 2017</td><td bgcolor="#ffffff">Aug 27 2018</td><td bgcolor="#ffffff">RNA</td><td bgcolor="#ffffff">1</td><td bgcolor="#ffffff">MDA231-LM2 breast cancer cells, treated with DMSO vehicle, 48h</td><td bgcolor="#ffffff">Homo sapiens</td><td bgcolor="#ffffff">cell line: MDA231-LM2</td><td bgcolor="#ffffff">tissue type: breast cancer</td><td bgcolor="#ffffff">treatment: DMSO_Vehicle</td><td bgcolor="#ffffff">MDA231-LM2 cells were seeded at a density of approximately 28.000 cells/cm2 and let to attach to the culture dish overnight. Then cells were treated with either a vehicle control (DMSO) or 5nM paclitaxel for 48h prior to lysis for RNA extraction.</td><td bgcolor="#ffffff">MDA231-LM2 cells were grown in DMEM GlutaMAX supplemented with 10% vol/vol fetal bovine serum (FBS), 100 IU/ml penicillin, 100 μg/ml streptomycin and 1 μg/ml amphotericin B.</td><td bgcolor="#ffffff">total RNA</td><td bgcolor="#ffffff">RNA extraction was performed using the RNEasy Kit (Qiagen) according to manufacturer's instructions</td><td bgcolor="#ffffff">biotin</td><td bgcolor="#ffffff">The Affymetrix Gene Chip 3´IVT PLUS Reagent Kit was used to generate labeled single-stranded cRNA from input amounts of 100ng total RNA. The label used was biotin.</td><td bgcolor="#ffffff">9606</td><td bgcolor="#ffffff">11 ug of fragmented and labeled cRNA were hybridized for 17 hours at 45°C on Affymetrix GeneChip® Human Genome U133 Plus 2.0 Arrays, according to user manual GeneChip® 3' IVT PLUS Reagent Kit, P/N 703210.</td><td bgcolor="#ffffff">Gene Expression Microarrays were scanned using the Affymetrix GeneChip® Scanner 3000 according to GeneChip® Expression Wash, Stain and Scan Manual for Cartridge Arrays  (P/N 702731)</td><td bgcolor="#ffffff">Gene expression of MDA231-LM2 cells cultured as a monolayer</td><td bgcolor="#ffffff">The raw data was processed using R and bioconductor package affy; normalization method is quantile normalization.</td><td bgcolor="#ffffff">GPL570</td><td bgcolor="#ffffff">Jacob,,Insua-Rodríguez</td><td bgcolor="#ffffff">j.insuarodriguez dkfz.de</td><td bgcolor="#ffffff">Oskarsson Lab</td><td bgcolor="#ffffff">Division of Stem Cells and Cancer</td><td bgcolor="#ffffff">German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ)</td><td bgcolor="#ffffff">Im Neuenheimer Feld 280</td><td bgcolor="#ffffff">Heidelberg</td><td bgcolor="#ffffff">Baden-Württemberg</td><td bgcolor="#ffffff">69120</td><td bgcolor="#ffffff">Germany</td><td bgcolor="#ffffff">ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2589nnn/GSM2589736/suppl/GSM2589736_DMSO_3.CEL.gz</td><td bgcolor="#ffffff">54675</td><td bgcolor="#ffffff">MDA231-LM2</td><td bgcolor="#ffffff">breast cancer</td><td bgcolor="#ffffff">DMSO_Vehicle</td></tr>
<tr>
<td bgcolor="#d0d0ff"><input type='checkbox' name='ReportObjectCheckBoxes' value='' onchange='checkboxEvent(4)'></td><td bgcolor="#e0e0f0">4</td><td bgcolor="#d0d0ff">GSM2589737</td><td bgcolor="#e0e0f0"></td><td bgcolor="#d0d0ff"></td><td bgcolor="#e0e0f0"></td><td bgcolor="#d0d0ff">MDA231-LM2_Paclitaxel, biological rep1</td><td bgcolor="#e0e0f0">GSM2589737</td><td bgcolor="#d0d0ff">Public on Aug 26 2018</td><td bgcolor="#e0e0f0">Apr 26 2017</td><td bgcolor="#d0d0ff">Aug 27 2018</td><td bgcolor="#e0e0f0">RNA</td><td bgcolor="#d0d0ff">1</td><td bgcolor="#e0e0f0">MDA231-LM2 breast cancer cells, treated with paclitaxel 5nM, 48h</td><td bgcolor="#d0d0ff">Homo sapiens</td><td bgcolor="#e0e0f0">cell line: MDA231-LM2</td><td bgcolor="#d0d0ff">tissue type: breast cancer</td><td bgcolor="#e0e0f0">treatment: Paclitaxel_5nM</td><td bgcolor="#d0d0ff">MDA231-LM2 cells were seeded at a density of approximately 28.000 cells/cm2 and let to attach to the culture dish overnight. Then cells were treated with either a vehicle control (DMSO) or 5nM paclitaxel for 48h prior to lysis for RNA extraction.</td><td bgcolor="#e0e0f0">MDA231-LM2 cells were grown in DMEM GlutaMAX supplemented with 10% vol/vol fetal bovine serum (FBS), 100 IU/ml penicillin, 100 μg/ml streptomycin and 1 μg/ml amphotericin B.</td><td bgcolor="#d0d0ff">total RNA</td><td bgcolor="#e0e0f0">RNA extraction was performed using the RNEasy Kit (Qiagen) according to manufacturer's instructions</td><td bgcolor="#d0d0ff">biotin</td><td bgcolor="#e0e0f0">The Affymetrix Gene Chip 3´IVT PLUS Reagent Kit was used to generate labeled single-stranded cRNA from input amounts of 100ng total RNA. The label used was biotin.</td><td bgcolor="#d0d0ff">9606</td><td bgcolor="#e0e0f0">11 ug of fragmented and labeled cRNA were hybridized for 17 hours at 45°C on Affymetrix GeneChip® Human Genome U133 Plus 2.0 Arrays, according to user manual GeneChip® 3' IVT PLUS Reagent Kit, P/N 703210.</td><td bgcolor="#d0d0ff">Gene Expression Microarrays were scanned using the Affymetrix GeneChip® Scanner 3000 according to GeneChip® Expression Wash, Stain and Scan Manual for Cartridge Arrays  (P/N 702731)</td><td bgcolor="#e0e0f0">Gene expression of MDA231-LM2 cells cultured as oncospheres</td><td bgcolor="#d0d0ff">The raw data was processed using R and bioconductor package affy; normalization method is quantile normalization.</td><td bgcolor="#e0e0f0">GPL570</td><td bgcolor="#d0d0ff">Jacob,,Insua-Rodríguez</td><td bgcolor="#e0e0f0">j.insuarodriguez dkfz.de</td><td bgcolor="#d0d0ff">Oskarsson Lab</td><td bgcolor="#e0e0f0">Division of Stem Cells and Cancer</td><td bgcolor="#d0d0ff">German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ)</td><td bgcolor="#e0e0f0">Im Neuenheimer Feld 280</td><td bgcolor="#d0d0ff">Heidelberg</td><td bgcolor="#e0e0f0">Baden-Württemberg</td><td bgcolor="#d0d0ff">69120</td><td bgcolor="#e0e0f0">Germany</td><td bgcolor="#d0d0ff">ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2589nnn/GSM2589737/suppl/GSM2589737_Paclitaxel_1.CEL.gz</td><td bgcolor="#e0e0f0">54675</td><td bgcolor="#d0d0ff">MDA231-LM2</td><td bgcolor="#e0e0f0">breast cancer</td><td bgcolor="#d0d0ff">Paclitaxel_5nM</td></tr>
<tr>
<td bgcolor="#ffffff"><input type='checkbox' name='ReportObjectCheckBoxes' value='' onchange='checkboxEvent(5)'></td><td bgcolor="#ffffff">5</td><td bgcolor="#ffffff">GSM2589738</td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff"></td><td bgcolor="#ffffff">MDA231-LM2_Paclitaxel, biological rep2</td><td bgcolor="#ffffff">GSM2589738</td><td bgcolor="#ffffff">Public on Aug 26 2018</td><td bgcolor="#ffffff">Apr 26 2017</td><td bgcolor="#ffffff">Aug 27 2018</td><td bgcolor="#ffffff">RNA</td><td bgcolor="#ffffff">1</td><td bgcolor="#ffffff">MDA231-LM2 breast cancer cells, treated with paclitaxel 5nM, 48h</td><td bgcolor="#ffffff">Homo sapiens</td><td bgcolor="#ffffff">cell line: MDA231-LM2</td><td bgcolor="#ffffff">tissue type: breast cancer</td><td bgcolor="#ffffff">treatment: Paclitaxel_5nM</td><td bgcolor="#ffffff">MDA231-LM2 cells were seeded at a density of approximately 28.000 cells/cm2 and let to attach to the culture dish overnight. Then cells were treated with either a vehicle control (DMSO) or 5nM paclitaxel for 48h prior to lysis for RNA extraction.</td><td bgcolor="#ffffff">MDA231-LM2 cells were grown in DMEM GlutaMAX supplemented with 10% vol/vol fetal bovine serum (FBS), 100 IU/ml penicillin, 100 μg/ml streptomycin and 1 μg/ml amphotericin B.</td><td bgcolor="#ffffff">total RNA</td><td bgcolor="#ffffff">RNA extraction was performed using the RNEasy Kit (Qiagen) according to manufacturer's instructions</td><td bgcolor="#ffffff">biotin</td><td bgcolor="#ffffff">The Affymetrix Gene Chip 3´IVT PLUS Reagent Kit was used to generate labeled single-stranded cRNA from input amounts of 100ng total RNA. The label used was biotin.</td><td bgcolor="#ffffff">9606</td><td bgcolor="#ffffff">11 ug of fragmented and labeled cRNA were hybridized for 17 hours at 45°C on Affymetrix GeneChip® Human Genome U133 Plus 2.0 Arrays, according to user manual GeneChip® 3' IVT PLUS Reagent Kit, P/N 703210.</td><td bgcolor="#ffffff">Gene Expression Microarrays were scanned using the Affymetrix GeneChip® Scanner 3000 according to GeneChip® Expression Wash, Stain and Scan Manual for Cartridge Arrays  (P/N 702731)</td><td bgcolor="#ffffff">Gene expression of MDA231-LM2 cells cultured as oncospheres</td><td bgcolor="#ffffff">The raw data was processed using R and bioconductor package affy; normalization method is quantile normalization.</td><td bgcolor="#ffffff">GPL570</td><td bgcolor="#ffffff">Jacob,,Insua-Rodríguez</td><td bgcolor="#ffffff">j.insuarodriguez dkfz.de</td><td bgcolor="#ffffff">Oskarsson Lab</td><td bgcolor="#ffffff">Division of Stem Cells and Cancer</td><td bgcolor="#ffffff">German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ)</td><td bgcolor="#ffffff">Im Neuenheimer Feld 280</td><td bgcolor="#ffffff">Heidelberg</td><td bgcolor="#ffffff">Baden-Württemberg</td><td bgcolor="#ffffff">69120</td><td bgcolor="#ffffff">Germany</td><td bgcolor="#ffffff">ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2589nnn/GSM2589738/suppl/GSM2589738_Paclitaxel_2.CEL.gz</td><td bgcolor="#ffffff">54675</td><td bgcolor="#ffffff">MDA231-LM2</td><td bgcolor="#ffffff">breast cancer</td><td bgcolor="#ffffff">Paclitaxel_5nM</td></tr>
<tr>
<td bgcolor="#d0d0ff"><input type='checkbox' name='ReportObjectCheckBoxes' value='' onchange='checkboxEvent(6)'></td><td bgcolor="#e0e0f0">6</td><td bgcolor="#d0d0ff">GSM2589739</td><td bgcolor="#e0e0f0"></td><td bgcolor="#d0d0ff"></td><td bgcolor="#e0e0f0"></td><td bgcolor="#d0d0ff">MDA231-LM2_Paclitaxel, biological rep3</td><td bgcolor="#e0e0f0">GSM2589739</td><td bgcolor="#d0d0ff">Public on Aug 26 2018</td><td bgcolor="#e0e0f0">Apr 26 2017</td><td bgcolor="#d0d0ff">Aug 27 2018</td><td bgcolor="#e0e0f0">RNA</td><td bgcolor="#d0d0ff">1</td><td bgcolor="#e0e0f0">MDA231-LM2 breast cancer cells, treated with paclitaxel 5nM, 48h</td><td bgcolor="#d0d0ff">Homo sapiens</td><td bgcolor="#e0e0f0">cell line: MDA231-LM2</td><td bgcolor="#d0d0ff">tissue type: breast cancer</td><td bgcolor="#e0e0f0">treatment: Paclitaxel_5nM</td><td bgcolor="#d0d0ff">MDA231-LM2 cells were seeded at a density of approximately 28.000 cells/cm2 and let to attach to the culture dish overnight. Then cells were treated with either a vehicle control (DMSO) or 5nM paclitaxel for 48h prior to lysis for RNA extraction.</td><td bgcolor="#e0e0f0">MDA231-LM2 cells were grown in DMEM GlutaMAX supplemented with 10% vol/vol fetal bovine serum (FBS), 100 IU/ml penicillin, 100 μg/ml streptomycin and 1 μg/ml amphotericin B.</td><td bgcolor="#d0d0ff">total RNA</td><td bgcolor="#e0e0f0">RNA extraction was performed using the RNEasy Kit (Qiagen) according to manufacturer's instructions</td><td bgcolor="#d0d0ff">biotin</td><td bgcolor="#e0e0f0">The Affymetrix Gene Chip 3´IVT PLUS Reagent Kit was used to generate labeled single-stranded cRNA from input amounts of 100ng total RNA. The label used was biotin.</td><td bgcolor="#d0d0ff">9606</td><td bgcolor="#e0e0f0">11 ug of fragmented and labeled cRNA were hybridized for 17 hours at 45°C on Affymetrix GeneChip® Human Genome U133 Plus 2.0 Arrays, according to user manual GeneChip® 3' IVT PLUS Reagent Kit, P/N 703210.</td><td bgcolor="#d0d0ff">Gene Expression Microarrays were scanned using the Affymetrix GeneChip® Scanner 3000 according to GeneChip® Expression Wash, Stain and Scan Manual for Cartridge Arrays  (P/N 702731)</td><td bgcolor="#e0e0f0">Gene expression of MDA231-LM2 cells cultured as oncospheres</td><td bgcolor="#d0d0ff">The raw data was processed using R and bioconductor package affy; normalization method is quantile normalization.</td><td bgcolor="#e0e0f0">GPL570</td><td bgcolor="#d0d0ff">Jacob,,Insua-Rodríguez</td><td bgcolor="#e0e0f0">j.insuarodriguez dkfz.de</td><td bgcolor="#d0d0ff">Oskarsson Lab</td><td bgcolor="#e0e0f0">Division of Stem Cells and Cancer</td><td bgcolor="#d0d0ff">German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ)</td><td bgcolor="#e0e0f0">Im Neuenheimer Feld 280</td><td bgcolor="#d0d0ff">Heidelberg</td><td bgcolor="#e0e0f0">Baden-Württemberg</td><td bgcolor="#d0d0ff">69120</td><td bgcolor="#e0e0f0">Germany</td><td bgcolor="#d0d0ff">ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2589nnn/GSM2589739/suppl/GSM2589739_Paclitaxel_3.CEL.gz</td><td bgcolor="#e0e0f0">54675</td><td bgcolor="#d0d0ff">MDA231-LM2</td><td bgcolor="#e0e0f0">breast cancer</td><td bgcolor="#d0d0ff">Paclitaxel_5nM</td></tr>
</table>
<br>The columns named *1, *2, ... indicate the calls from the different outlier detection methods:<OL><LI> outlier detection by <a href="#hm">Distances between arrays</a></LI><LI> outlier detection by <a href="#box">Boxplots</a></LI><LI> outlier detection by <a href="#ma">MA plots</a></LI></OL>The outlier detection criteria are explained below in the respective sections. Arrays that were called outliers by at least one criterion are marked by checkbox selection in this table, and are indicated by highlighted lines or points in some of the plots below. By clicking the checkboxes in the table, or on the corresponding points/lines in the plots, you can modify the selection. To reset the selection, reload the HTML page in your browser.<br><br>At the scope covered by this software, outlier detection is a poorly defined question, and there is no 'right' or 'wrong' answer. These are hints which are intended to be followed up manually. If you want to automate outlier detection, you need to limit the scope to a particular platform and experimental design, and then choose and calibrate the metrics used.<br></div><hr><h2><a name= 'S1'>Section 1: Between array comparison</a></h2>

<a name="hm" id="hm-h" href="javascript: toggle('hm')" style="text-decoration:none;font-weight:bold;font-size:larger">- Figure 1: Distances between arrays.</a><br>

<div id="hm-b" style="display:block">
<img src="hm.png" border="0" alt="hm.png" id="Figls:hm"></img><br>
<b>Figure 1</b><a href="hm.pdf"> (PDF file)</a> shows a false color heatmap of the distances between arrays. The color scale is chosen to cover the range of distances encountered in the dataset. Patterns in this plot can indicate clustering of the arrays either because of intended biological or unintended experimental factors (batch effects). The distance <i>d<sub>ab</sub></i> between two arrays <i>a</i> and <i>b</i> is computed as the mean absolute difference (L<sub>1</sub>-distance) between the data of the arrays (using the data from all probes without filtering). In formula, <i>d<sub>ab</sub></i> = mean | <i>M<sub>ai</sub> - M<sub>bi</sub></i> |, where <i>M<sub>ai</sub></i> is the value of the <i>i</i>-th probe on the <i>a</i>-th array. Outlier detection was performed by looking for arrays for which the sum of the distances to all other arrays, <i>S<sub>a</sub></i> = &Sigma;<sub><i>b</i></sub> <i>d<sub>ab</sub></i> was exceptionally large. No such arrays were detected.<br><br><br>
</div>

<a name="out hm" id="out hm-h" href="javascript: toggle('out hm')" style="text-decoration:none;font-weight:bold;font-size:larger">+ Figure 2: Outlier detection for Distances between arrays.</a><br>

<div id="out hm-b" style="display:none">
<img src="out hm.png" border="0" alt="out hm.png" id="Figls:out hm"></img><br>
<b>Figure 2</b><a href="out hm.pdf"> (PDF file)</a> shows a bar chart of the sum of distances to other arrays <i>S<sub>a</sub></i>, the outlier detection criterion from the previous figure. The bars are shown in the original order of the arrays. Based on the distribution of the values across all arrays, a threshold of 239 was determined, which is indicated by the vertical line. None of the arrays exceeded the threshold and was considered an outlier.<br><br><br>
</div>

<a name="pca" id="pca-h" href="javascript: toggle('pca')" style="text-decoration:none;font-weight:bold;font-size:larger">- Figure 3: Principal Component Analysis.</a><br>

<div id="pca-b" style="display:block">
<table border="1">
<tr>
<td><?xml version="1.0" encoding="ISO8859-1"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="323.27px" height="409.67px" viewBox="0 0 323.27 409.67" version="1.1" id="Fig:pca">
  <metadata xmlns:gridsvg="http://www.stat.auckland.ac.nz/~paul/R/gridSVG/">
    <gridsvg:generator name="gridSVG" version="1.7-1" time="2020-04-25 21:30:18"/>
    <gridsvg:argument name="name" value="C:/Bea/Master/Datos omicos Bea/PEC1/arrayQM/pca.svg"/>
    <gridsvg:argument name="exportCoords" value="none"/>
    <gridsvg:argument name="exportMappings" value="none"/>
    <gridsvg:argument name="exportJS" value="none"/>
    <gridsvg:argument name="res" value="72"/>
    <gridsvg:argument name="prefix" value="Fig:pca"/>
    <gridsvg:argument name="addClasses" value="FALSE"/>
    <gridsvg:argument name="indent" value="TRUE"/>
    <gridsvg:argument name="htmlWrapper" value="FALSE"/>
    <gridsvg:argument name="usePaths" value="none"/>
    <gridsvg:argument name="uniqueNames" value="TRUE"/>
    <gridsvg:separator name="id.sep" value="."/>
    <gridsvg:separator name="gPath.sep" value="::"/>
    <gridsvg:separator name="vpPath.sep" value="::"/>
  </metadata>
  <g transform="translate(0, 409.67) scale(1, -1)">
    <g id="Fig:pcagridSVG" fill="none" stroke="rgb(0,0,0)" stroke-dasharray="none" stroke-width="0.75" font-size="12" font-family="Helvetica, Arial, FreeSans, Liberation Sans, Nimbus Sans L, sans-serif" opacity="1" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" stroke-opacity="1" fill-opacity="0" font-weight="normal" font-style="normal">
      <defs>
        <symbol id="Fig:pcagridSVG.pch19" viewBox="-5 -5 10 10" overflow="visible">
          <circle cx="0" cy="0" r="3.75"/>
        </symbol>
      </defs>
      <g id="Fig:pcaplot_01.background.1">
        <rect id="Fig:pcaplot_01.background.1.1" x="0" y="0" width="323.27" height="409.67" transform="" fill="none" stroke="none" stroke-opacity="0" fill-opacity="0"/>
      </g>
      <g id="Fig:pcaplot_01.toplevel.vp.1" font-size="12">
        <g id="Fig:pcaplot_01.main.vp.1">
          <g id="Fig:pcaGRID.VP.119.1">
            <g id="Fig:pca.key.frame.1">
              <g id="Fig:pcaGRID.VP.120.1">
                <g id="Fig:pcaGRID.cellGrob.128.1">
                  <g id="Fig:pca.key.background.1">
                    <rect id="Fig:pca.key.background.1.1" x="7.54" y="286.99" width="308.18" height="68.98" transform="" fill="none" opacity="1" stroke="none" stroke-opacity="0" fill-opacity="0"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.121.1">
                <g id="Fig:pcaGRID.cellGrob.129.1">
                  <g id="Fig:pca.key.rect.1.1.1">
                    <rect id="Fig:pca.key.rect.1.1.1.1" x="14.22" y="344.47" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(228,26,28)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.122.1">
                <g id="Fig:pcaGRID.cellGrob.130.1">
                  <g id="Fig:pca.key.rect.1.2.1">
                    <rect id="Fig:pca.key.rect.1.2.1.1" x="14.22" y="332.98" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(55,126,184)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.123.1">
                <g id="Fig:pcaGRID.cellGrob.131.1">
                  <g id="Fig:pca.key.rect.1.3.1">
                    <rect id="Fig:pca.key.rect.1.3.1.1" x="14.22" y="321.48" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(77,175,74)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.124.1">
                <g id="Fig:pcaGRID.cellGrob.132.1">
                  <g id="Fig:pca.key.rect.1.4.1">
                    <rect id="Fig:pca.key.rect.1.4.1.1" x="14.22" y="309.98" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(152,78,163)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.125.1">
                <g id="Fig:pcaGRID.cellGrob.133.1">
                  <g id="Fig:pca.key.rect.1.5.1">
                    <rect id="Fig:pca.key.rect.1.5.1.1" x="14.22" y="298.49" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(255,127,0)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.126.1">
                <g id="Fig:pcaGRID.cellGrob.134.1">
                  <g id="Fig:pca.key.rect.1.6.1">
                    <rect id="Fig:pca.key.rect.1.6.1.1" x="14.22" y="286.99" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(255,255,51)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.127.1">
                <g id="Fig:pcaGRID.cellGrob.135.1">
                  <g id="Fig:pca.key.text.2.1.1">
                    <g id="Fig:pca.key.text.2.1.1.1" transform="translate(60.92, 350.22)" stroke-width="0.1">
                      <g id="Fig:pca.key.text.2.1.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:pca.key.text.2.1.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:pca.key.text.2.1.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_DMSO_Vehicle, biological rep1</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.128.1">
                <g id="Fig:pcaGRID.cellGrob.136.1">
                  <g id="Fig:pca.key.text.2.2.1">
                    <g id="Fig:pca.key.text.2.2.1.1" transform="translate(60.92, 338.72)" stroke-width="0.1">
                      <g id="Fig:pca.key.text.2.2.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:pca.key.text.2.2.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:pca.key.text.2.2.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_DMSO_Vehicle, biological rep2</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.129.1">
                <g id="Fig:pcaGRID.cellGrob.137.1">
                  <g id="Fig:pca.key.text.2.3.1">
                    <g id="Fig:pca.key.text.2.3.1.1" transform="translate(60.92, 327.23)" stroke-width="0.1">
                      <g id="Fig:pca.key.text.2.3.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:pca.key.text.2.3.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:pca.key.text.2.3.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_DMSO_Vehicle, biological rep3</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.130.1">
                <g id="Fig:pcaGRID.cellGrob.138.1">
                  <g id="Fig:pca.key.text.2.4.1">
                    <g id="Fig:pca.key.text.2.4.1.1" transform="translate(60.92, 315.73)" stroke-width="0.1">
                      <g id="Fig:pca.key.text.2.4.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:pca.key.text.2.4.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:pca.key.text.2.4.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_Paclitaxel, biological rep1</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.131.1">
                <g id="Fig:pcaGRID.cellGrob.139.1">
                  <g id="Fig:pca.key.text.2.5.1">
                    <g id="Fig:pca.key.text.2.5.1.1" transform="translate(60.92, 304.24)" stroke-width="0.1">
                      <g id="Fig:pca.key.text.2.5.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:pca.key.text.2.5.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:pca.key.text.2.5.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_Paclitaxel, biological rep2</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:pcaGRID.VP.132.1">
                <g id="Fig:pcaGRID.cellGrob.140.1">
                  <g id="Fig:pca.key.text.2.6.1">
                    <g id="Fig:pca.key.text.2.6.1.1" transform="translate(60.92, 292.74)" stroke-width="0.1">
                      <g id="Fig:pca.key.text.2.6.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:pca.key.text.2.6.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:pca.key.text.2.6.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_Paclitaxel, biological rep3</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:pcaplot_01.xlab.vp.1">
          <g id="Fig:pcaplot_01.xlab.1">
            <g id="Fig:pcaplot_01.xlab.1.1" transform="translate(182.76, 64.01)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.xlab.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.xlab.1.1.text" text-anchor="middle" opacity="1" stroke="rgb(0,0,0)" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.xlab.1.1.tspan.1" dy="4.31" x="0">PC1</tspan>
                </text>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:pcaplot_01.ylab.vp.1">
          <g id="Fig:pcaplot_01.ylab.1">
            <g id="Fig:pcaplot_01.ylab.1.1" transform="translate(10.31, 178.25)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ylab.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ylab.1.1.text" transform="rotate(-90)" text-anchor="middle" opacity="1" stroke="rgb(0,0,0)" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ylab.1.1.tspan.1" dy="4.31" x="0">PC2</tspan>
                </text>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:pcaplot_01.figure.vp.1"/>
        <defs>
          <clipPath id="Fig:pcaplot_01.panel.1.1.vp.1.clipPath">
            <rect x="71.26" y="98.51" width="223" height="159.48" fill="none" stroke="none"/>
          </clipPath>
        </defs>
        <g id="Fig:pcaplot_01.panel.1.1.vp.1" clip-path="url(#Fig:pcaplot_01.panel.1.1.vp.1.clipPath)"/>
        <g id="Fig:pcaplot_01.strip.1.1.off.vp.1">
          <g id="Fig:pcaplot_01.ticks.top.panel.1.1.1">
            <polyline id="Fig:pcaplot_01.ticks.top.panel.1.1.1.1" points="126.42,257.98 126.42,263.65" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.top.panel.1.1.1.2" points="184.61,257.98 184.61,263.65" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.top.panel.1.1.1.3" points="242.81,257.98 242.81,263.65" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
          </g>
        </g>
        <g id="Fig:pcaplot_01.strip.left.1.1.off.vp.1">
          <g id="Fig:pcaplot_01.ticks.left.panel.1.1.1">
            <polyline id="Fig:pcaplot_01.ticks.left.panel.1.1.1.1" points="71.26,121.52 65.59,121.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.left.panel.1.1.1.2" points="71.26,150.62 65.59,150.62" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.left.panel.1.1.1.3" points="71.26,179.72 65.59,179.72" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.left.panel.1.1.1.4" points="71.26,208.82 65.59,208.82" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.left.panel.1.1.1.5" points="71.26,237.91 65.59,237.91" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1">
            <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.1" transform="translate(59.93, 121.52)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.1.text" text-anchor="end" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.1.tspan.1" dy="3.59" x="0">-10000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.2" transform="translate(59.93, 150.62)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.2.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.2.text" text-anchor="end" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.2.tspan.1" dy="3.59" x="0">-5000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.3" transform="translate(59.93, 179.72)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.3.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.3.text" text-anchor="end" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.3.tspan.1" dy="3.59" x="0">0</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.4" transform="translate(59.93, 208.82)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.4.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.4.text" text-anchor="end" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.4.tspan.1" dy="3.59" x="0">5000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.5" transform="translate(59.93, 237.91)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.5.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.5.text" text-anchor="end" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.left.panel.1.1.1.5.tspan.1" dy="3.59" x="0">10000</tspan>
                </text>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:pcaplot_01.panel.1.1.off.vp.1">
          <g id="Fig:pcaplot_01.ticks.bottom.panel.1.1.1">
            <polyline id="Fig:pcaplot_01.ticks.bottom.panel.1.1.1.1" points="126.42,98.51 126.42,92.84" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.bottom.panel.1.1.1.2" points="184.61,98.51 184.61,92.84" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.bottom.panel.1.1.1.3" points="242.81,98.51 242.81,92.84" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1">
            <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.1" transform="translate(126.42, 87.17)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.1.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.1.tspan.1" dy="7.18" x="0">-10000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.2" transform="translate(184.61, 87.17)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.2.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.2.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.2.tspan.1" dy="7.18" x="0">0</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.3" transform="translate(242.81, 87.17)" stroke-width="0.1">
              <g id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.3.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.3.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:pcaplot_01.ticklabels.bottom.panel.1.1.1.3.tspan.1" dy="7.18" x="0">10000</tspan>
                </text>
              </g>
            </g>
          </g>
          <g id="Fig:pcaplot_01.ticks.right.panel.1.1.1">
            <polyline id="Fig:pcaplot_01.ticks.right.panel.1.1.1.1" points="294.26,121.52 299.93,121.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.right.panel.1.1.1.2" points="294.26,150.62 299.93,150.62" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.right.panel.1.1.1.3" points="294.26,179.72 299.93,179.72" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.right.panel.1.1.1.4" points="294.26,208.82 299.93,208.82" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:pcaplot_01.ticks.right.panel.1.1.1.5" points="294.26,237.91 299.93,237.91" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
          </g>
        </g>
        <defs>
          <clipPath id="Fig:pcaplot_01.panel.1.1.vp.2.clipPath">
            <rect x="71.26" y="98.51" width="223" height="159.48" fill="none" stroke="none"/>
          </clipPath>
        </defs>
        <g id="Fig:pcaplot_01.panel.1.1.vp.2" clip-path="url(#Fig:pcaplot_01.panel.1.1.vp.2.clipPath)">
          <g id="Fig:pcaplot_01.xyplot.points.panel.1.1.1">
            <use id="Fig:pcaplot_01.xyplot.points.panel.1.1.1.1" href="#Fig:pcagridSVG.pch19" x="84.96" y="220.32" width="8" height="8" transform=" translate(-4,-4)" stroke="rgb(228,26,28)" stroke-width="0.94" opacity="1" fill="rgb(228,26,28)" font-size="8" stroke-opacity="1" fill-opacity="1" font-weight="normal" font-style="normal"/>
            <use id="Fig:pcaplot_01.xyplot.points.panel.1.1.1.2" href="#Fig:pcagridSVG.pch19" x="104.87" y="110.02" width="8" height="8" transform=" translate(-4,-4)" stroke="rgb(55,126,184)" stroke-width="0.94" opacity="1" fill="rgb(55,126,184)" font-size="8" stroke-opacity="1" fill-opacity="1" font-weight="normal" font-style="normal"/>
            <use id="Fig:pcaplot_01.xyplot.points.panel.1.1.1.3" href="#Fig:pcagridSVG.pch19" x="110.97" y="192.91" width="8" height="8" transform=" translate(-4,-4)" stroke="rgb(77,175,74)" stroke-width="0.94" opacity="1" fill="rgb(77,175,74)" font-size="8" stroke-opacity="1" fill-opacity="1" font-weight="normal" font-style="normal"/>
            <use id="Fig:pcaplot_01.xyplot.points.panel.1.1.1.4" href="#Fig:pcagridSVG.pch19" x="274.13" y="198.56" width="8" height="8" transform=" translate(-4,-4)" stroke="rgb(152,78,163)" stroke-width="0.94" opacity="1" fill="rgb(152,78,163)" font-size="8" stroke-opacity="1" fill-opacity="1" font-weight="normal" font-style="normal"/>
            <use id="Fig:pcaplot_01.xyplot.points.panel.1.1.1.5" href="#Fig:pcagridSVG.pch19" x="280.57" y="108.3" width="8" height="8" transform=" translate(-4,-4)" stroke="rgb(255,127,0)" stroke-width="0.94" opacity="1" fill="rgb(255,127,0)" font-size="8" stroke-opacity="1" fill-opacity="1" font-weight="normal" font-style="normal"/>
            <use id="Fig:pcaplot_01.xyplot.points.panel.1.1.1.6" href="#Fig:pcagridSVG.pch19" x="252.17" y="248.19" width="8" height="8" transform=" translate(-4,-4)" stroke="rgb(255,255,51)" stroke-width="0.94" opacity="1" fill="rgb(255,255,51)" font-size="8" stroke-opacity="1" fill-opacity="1" font-weight="normal" font-style="normal"/>
          </g>
        </g>
        <g id="Fig:pcaplot_01.panel.1.1.off.vp.2">
          <g id="Fig:pcaplot_01.border.panel.1.1.1">
            <rect id="Fig:pcaplot_01.border.panel.1.1.1.1" x="71.26" y="98.51" width="223" height="159.48" transform="" stroke="rgb(0,0,0)" stroke-dasharray="none" stroke-width="0.75" opacity="1" fill="none" stroke-opacity="1" fill-opacity="0"/>
          </g>
        </g>
        <g id="Fig:pcaplot_01..1"/>
      </g>
    </g>
  </g>
</svg></td><td><table id='Tab:pca'><tr bgcolor='#d0d0ff'><td>array</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>sampleNames</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>title</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>geo_accession</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>status</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>submission_date</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>last_update_date</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>type</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>channel_count</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>source_name_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>organism_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>characteristics_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>characteristics_ch1.1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>characteristics_ch1.2</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>treatment_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>growth_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>molecule_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>extract_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>label_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>label_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>taxid_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>hyb_protocol</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>scan_protocol</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>description</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>data_processing</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>platform_id</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_name</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_email</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_laboratory</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_department</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_institute</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_address</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_city</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_state</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_zip/postal_code</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_country</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>supplementary_file</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>data_row_count</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>cell line:ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>tissue type:ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>treatment:ch1</td><td style='font-weight:bold'></td></tr>
</table></td></tr>
</table>
<br>
<b>Figure 3</b><a href="pca.pdf"> (PDF file)</a> shows a scatterplot of the arrays along the first two principal components. You can use this plot to explore if the arrays cluster, and whether this is according to an intended experimental factor, or according to unintended causes such as batch effects. Move the mouse over the points to see the sample names.<BR>Principal component analysis is a dimension reduction and visualisation technique that is here used to project the multivariate data vector of each array into a two-dimensional plot, such that the spatial arrangement of the points in the plot reflects the overall data (dis)similarity between the arrays.<br><br><br>
</div><hr><h2><a name= 'S2'>Section 2: Array intensity distributions</a></h2>

<a name="box" id="box-h" href="javascript: toggle('box')" style="text-decoration:none;font-weight:bold;font-size:larger">- Figure 4: Boxplots.</a><br>

<div id="box-b" style="display:block">
<img src="box.png" border="0" alt="box.png" id="Figls:box"></img><br>
<b>Figure 4</b><a href="box.pdf"> (PDF file)</a> shows boxplots representing summaries of the signal intensity distributions of the arrays. Each box corresponds to one array. Typically, one expects the boxes to have similar positions and widths. If the distribution of an array is very different from the others, this may indicate an experimental problem. Outlier detection was performed by computing the Kolmogorov-Smirnov statistic <i>K<sub>a</sub></i> between each array's distribution and the distribution of the pooled data.<br><br><br>
</div>

<a name="out box" id="out box-h" href="javascript: toggle('out box')" style="text-decoration:none;font-weight:bold;font-size:larger">+ Figure 5: Outlier detection for Boxplots.</a><br>

<div id="out box-b" style="display:none">
<img src="out box.png" border="0" alt="out box.png" id="Figls:out box"></img><br>
<b>Figure 5</b><a href="out box.pdf"> (PDF file)</a> shows a bar chart of the Kolmogorov-Smirnov statistic <i>K<sub>a</sub></i>, the outlier detection criterion from the previous figure. The bars are shown in the original order of the arrays. Based on the distribution of the values across all arrays, a threshold of 0.0145 was determined, which is indicated by the vertical line. None of the arrays exceeded the threshold and was considered an outlier.<br><br><br>
</div>

<a name="dens" id="dens-h" href="javascript: toggle('dens')" style="text-decoration:none;font-weight:bold;font-size:larger">- Figure 6: Density plots.</a><br>

<div id="dens-b" style="display:block">
<table border="1">
<tr>
<td><?xml version="1.0" encoding="ISO8859-1"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="288px" height="338.4px" viewBox="0 0 288 338.4" version="1.1" id="Fig:dens">
  <metadata xmlns:gridsvg="http://www.stat.auckland.ac.nz/~paul/R/gridSVG/">
    <gridsvg:generator name="gridSVG" version="1.7-1" time="2020-04-25 21:30:19"/>
    <gridsvg:argument name="name" value="C:/Bea/Master/Datos omicos Bea/PEC1/arrayQM/dens.svg"/>
    <gridsvg:argument name="exportCoords" value="none"/>
    <gridsvg:argument name="exportMappings" value="none"/>
    <gridsvg:argument name="exportJS" value="none"/>
    <gridsvg:argument name="res" value="72"/>
    <gridsvg:argument name="prefix" value="Fig:dens"/>
    <gridsvg:argument name="addClasses" value="FALSE"/>
    <gridsvg:argument name="indent" value="TRUE"/>
    <gridsvg:argument name="htmlWrapper" value="FALSE"/>
    <gridsvg:argument name="usePaths" value="none"/>
    <gridsvg:argument name="uniqueNames" value="TRUE"/>
    <gridsvg:separator name="id.sep" value="."/>
    <gridsvg:separator name="gPath.sep" value="::"/>
    <gridsvg:separator name="vpPath.sep" value="::"/>
  </metadata>
  <g transform="translate(0, 338.4) scale(1, -1)">
    <g id="Fig:densgridSVG" fill="none" stroke="rgb(0,0,0)" stroke-dasharray="none" stroke-width="0.75" font-size="12" font-family="Helvetica, Arial, FreeSans, Liberation Sans, Nimbus Sans L, sans-serif" opacity="1" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" stroke-opacity="1" fill-opacity="0" font-weight="normal" font-style="normal">
      <g id="Fig:densplot_01.background.1">
        <rect id="Fig:densplot_01.background.1.1" x="0" y="0" width="288" height="338.4" transform="" fill="none" stroke="none" stroke-opacity="0" fill-opacity="0"/>
      </g>
      <g id="Fig:densplot_01.toplevel.vp.1" font-size="12">
        <g id="Fig:densplot_01.main.vp.1">
          <g id="Fig:densGRID.VP.147.1">
            <g id="Fig:dens.key.frame.1">
              <g id="Fig:densGRID.VP.148.1">
                <g id="Fig:densGRID.cellGrob.154.1">
                  <g id="Fig:dens.key.background.1">
                    <rect id="Fig:dens.key.background.1.1" x="-10.09" y="263.42" width="308.18" height="68.98" transform="" fill="none" opacity="1" stroke="none" stroke-opacity="0" fill-opacity="0"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.149.1">
                <g id="Fig:densGRID.cellGrob.155.1">
                  <g id="Fig:dens.key.rect.1.1.1">
                    <rect id="Fig:dens.key.rect.1.1.1.1" x="-3.42" y="320.9" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(228,26,28)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.150.1">
                <g id="Fig:densGRID.cellGrob.156.1">
                  <g id="Fig:dens.key.rect.1.2.1">
                    <rect id="Fig:dens.key.rect.1.2.1.1" x="-3.42" y="309.41" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(55,126,184)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.151.1">
                <g id="Fig:densGRID.cellGrob.157.1">
                  <g id="Fig:dens.key.rect.1.3.1">
                    <rect id="Fig:dens.key.rect.1.3.1.1" x="-3.42" y="297.91" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(77,175,74)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.152.1">
                <g id="Fig:densGRID.cellGrob.158.1">
                  <g id="Fig:dens.key.rect.1.4.1">
                    <rect id="Fig:dens.key.rect.1.4.1.1" x="-3.42" y="286.42" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(152,78,163)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.153.1">
                <g id="Fig:densGRID.cellGrob.159.1">
                  <g id="Fig:dens.key.rect.1.5.1">
                    <rect id="Fig:dens.key.rect.1.5.1.1" x="-3.42" y="274.92" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(255,127,0)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.154.1">
                <g id="Fig:densGRID.cellGrob.160.1">
                  <g id="Fig:dens.key.rect.1.6.1">
                    <rect id="Fig:dens.key.rect.1.6.1.1" x="-3.42" y="263.42" width="33.36" height="11.5" transform="" opacity="1" fill="rgb(255,255,51)" stroke="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1"/>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.155.1">
                <g id="Fig:densGRID.cellGrob.161.1">
                  <g id="Fig:dens.key.text.2.1.1">
                    <g id="Fig:dens.key.text.2.1.1.1" transform="translate(43.28, 326.65)" stroke-width="0.1">
                      <g id="Fig:dens.key.text.2.1.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:dens.key.text.2.1.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:dens.key.text.2.1.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_DMSO_Vehicle, biological rep1</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.156.1">
                <g id="Fig:densGRID.cellGrob.162.1">
                  <g id="Fig:dens.key.text.2.2.1">
                    <g id="Fig:dens.key.text.2.2.1.1" transform="translate(43.28, 315.16)" stroke-width="0.1">
                      <g id="Fig:dens.key.text.2.2.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:dens.key.text.2.2.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:dens.key.text.2.2.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_DMSO_Vehicle, biological rep2</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.157.1">
                <g id="Fig:densGRID.cellGrob.163.1">
                  <g id="Fig:dens.key.text.2.3.1">
                    <g id="Fig:dens.key.text.2.3.1.1" transform="translate(43.28, 303.66)" stroke-width="0.1">
                      <g id="Fig:dens.key.text.2.3.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:dens.key.text.2.3.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:dens.key.text.2.3.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_DMSO_Vehicle, biological rep3</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.158.1">
                <g id="Fig:densGRID.cellGrob.164.1">
                  <g id="Fig:dens.key.text.2.4.1">
                    <g id="Fig:dens.key.text.2.4.1.1" transform="translate(43.28, 292.16)" stroke-width="0.1">
                      <g id="Fig:dens.key.text.2.4.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:dens.key.text.2.4.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:dens.key.text.2.4.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_Paclitaxel, biological rep1</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.159.1">
                <g id="Fig:densGRID.cellGrob.165.1">
                  <g id="Fig:dens.key.text.2.5.1">
                    <g id="Fig:dens.key.text.2.5.1.1" transform="translate(43.28, 280.67)" stroke-width="0.1">
                      <g id="Fig:dens.key.text.2.5.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:dens.key.text.2.5.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:dens.key.text.2.5.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_Paclitaxel, biological rep2</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <g id="Fig:densGRID.VP.160.1">
                <g id="Fig:densGRID.cellGrob.166.1">
                  <g id="Fig:dens.key.text.2.6.1">
                    <g id="Fig:dens.key.text.2.6.1.1" transform="translate(43.28, 269.17)" stroke-width="0.1">
                      <g id="Fig:dens.key.text.2.6.1.1.scale" transform="scale(1, -1)">
                        <text x="0" y="0" id="Fig:dens.key.text.2.6.1.1.text" text-anchor="start" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                          <tspan id="Fig:dens.key.text.2.6.1.1.tspan.1" dy="4.31" x="0">MDA231-LM2_Paclitaxel, biological rep3</tspan>
                        </text>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:densplot_01.ylab.vp.1">
          <g id="Fig:densplot_01.ylab.1">
            <g id="Fig:densplot_01.ylab.1.1" transform="translate(10.31, 146.81)" stroke-width="0.1">
              <g id="Fig:densplot_01.ylab.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ylab.1.1.text" transform="rotate(-90)" text-anchor="middle" opacity="1" stroke="rgb(0,0,0)" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="12" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ylab.1.1.tspan.1" dy="4.31" x="0">Density</tspan>
                </text>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:densplot_01.figure.vp.1"/>
        <defs>
          <clipPath id="Fig:densplot_01.panel.1.1.vp.1.clipPath">
            <rect x="44.8" y="42.19" width="231.2" height="209.24" fill="none" stroke="none"/>
          </clipPath>
        </defs>
        <g id="Fig:densplot_01.panel.1.1.vp.1" clip-path="url(#Fig:densplot_01.panel.1.1.vp.1.clipPath)"/>
        <g id="Fig:densplot_01.strip.1.1.off.vp.1"/>
        <g id="Fig:densplot_01.strip.left.1.1.off.vp.1">
          <g id="Fig:densplot_01.ticks.left.panel.1.1.1">
            <polyline id="Fig:densplot_01.ticks.left.panel.1.1.1.1" points="44.8,55.04 39.13,55.04" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.left.panel.1.1.1.2" points="44.8,93.27 39.13,93.27" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.left.panel.1.1.1.3" points="44.8,131.5 39.13,131.5" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.left.panel.1.1.1.4" points="44.8,169.74 39.13,169.74" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.left.panel.1.1.1.5" points="44.8,207.97 39.13,207.97" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.left.panel.1.1.1.6" points="44.8,246.21 39.13,246.21" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1">
            <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.1" transform="translate(33.47, 55.04)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.left.panel.1.1.1.1.text" transform="rotate(-90)" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.left.panel.1.1.1.1.tspan.1" dy="0" x="0">0.000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.2" transform="translate(33.47, 93.27)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.2.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.left.panel.1.1.1.2.text" transform="rotate(-90)" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.left.panel.1.1.1.2.tspan.1" dy="0" x="0">0.002</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.3" transform="translate(33.47, 131.5)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.3.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.left.panel.1.1.1.3.text" transform="rotate(-90)" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.left.panel.1.1.1.3.tspan.1" dy="0" x="0">0.004</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.4" transform="translate(33.47, 169.74)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.4.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.left.panel.1.1.1.4.text" transform="rotate(-90)" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.left.panel.1.1.1.4.tspan.1" dy="0" x="0">0.006</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.5" transform="translate(33.47, 207.97)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.5.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.left.panel.1.1.1.5.text" transform="rotate(-90)" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.left.panel.1.1.1.5.tspan.1" dy="0" x="0">0.008</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.6" transform="translate(33.47, 246.21)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.left.panel.1.1.1.6.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.left.panel.1.1.1.6.text" transform="rotate(-90)" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.left.panel.1.1.1.6.tspan.1" dy="0" x="0">0.010</tspan>
                </text>
              </g>
            </g>
          </g>
        </g>
        <g id="Fig:densplot_01.panel.1.1.off.vp.1">
          <g id="Fig:densplot_01.ticks.bottom.panel.1.1.1">
            <polyline id="Fig:densplot_01.ticks.bottom.panel.1.1.1.1" points="58.79,42.19 58.79,36.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.bottom.panel.1.1.1.2" points="108,42.19 108,36.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.bottom.panel.1.1.1.3" points="157.21,42.19 157.21,36.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.bottom.panel.1.1.1.4" points="206.42,42.19 206.42,36.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
            <polyline id="Fig:densplot_01.ticks.bottom.panel.1.1.1.5" points="255.63,42.19 255.63,36.52" stroke="rgb(0,0,0)" opacity="1" stroke-dasharray="none" stroke-width="0.75" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1">
            <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.1" transform="translate(58.79, 30.85)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.1.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.1.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.1.tspan.1" dy="7.18" x="0">0</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.2" transform="translate(108, 30.85)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.2.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.2.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.2.tspan.1" dy="7.18" x="0">1000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.3" transform="translate(157.21, 30.85)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.3.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.3.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.3.tspan.1" dy="7.18" x="0">2000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.4" transform="translate(206.42, 30.85)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.4.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.4.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.4.tspan.1" dy="7.18" x="0">3000</tspan>
                </text>
              </g>
            </g>
            <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.5" transform="translate(255.63, 30.85)" stroke-width="0.1">
              <g id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.5.scale" transform="scale(1, -1)">
                <text x="0" y="0" id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.5.text" text-anchor="middle" stroke="rgb(0,0,0)" opacity="1" fill="rgb(0,0,0)" stroke-opacity="1" fill-opacity="1" font-size="9.6" font-weight="normal" font-style="normal">
                  <tspan id="Fig:densplot_01.ticklabels.bottom.panel.1.1.1.5.tspan.1" dy="7.18" x="0">4000</tspan>
                </text>
              </g>
            </g>
          </g>
        </g>
        <defs>
          <clipPath id="Fig:densplot_01.panel.1.1.vp.2.clipPath">
            <rect x="44.8" y="42.19" width="231.2" height="209.24" fill="none" stroke="none"/>
          </clipPath>
        </defs>
        <g id="Fig:densplot_01.panel.1.1.vp.2" clip-path="url(#Fig:densplot_01.panel.1.1.vp.2.clipPath)">
          <g id="Fig:densplot_01.xyplot.lines.group.1.panel.1.1.1">
            <polyline id="Fig:densplot_01.xyplot.lines.group.1.panel.1.1.1.1" points="59.18,55.11 59.58,55.43 59.97,56.84 60.37,61.84 60.77,75.01 61.16,101.37 61.56,141.29 61.96,186.39 62.35,222.49 62.75,238.58 63.15,233.21 63.54,213.03 63.94,186.9 64.34,161.43 64.73,139.84 65.13,122.84 65.53,109.93 65.92,100.25 66.32,92.98 66.71,87.49 67.11,83.37 67.51,80.26 67.9,77.88 68.3,75.97 68.7,74.41 69.09,73.11 69.49,71.97 69.89,70.9 70.28,69.85 70.68,68.86 71.08,67.98 71.47,67.2 71.87,66.52 72.27,65.99 72.66,65.66 73.06,65.5 73.46,65.41 73.85,65.23 74.25,64.89 74.64,64.41 75.04,63.87 75.44,63.36 75.83,62.94 76.23,62.68 76.63,62.57 77.02,62.52 77.42,62.42 77.82,62.24 78.21,62 78.61,61.76 79.01,61.56 79.4,61.4 79.8,61.26 80.2,61.13 80.59,60.99 80.99,60.82 81.39,60.63 81.78,60.42 82.18,60.21 82.57,60.01 82.97,59.86 83.37,59.77 83.76,59.75 84.16,59.77 84.56,59.78 84.95,59.78 85.35,59.76 85.75,59.72 86.14,59.64 86.54,59.52 86.94,59.37 87.33,59.22 87.73,59.11 88.13,59.02 88.52,58.94 88.92,58.85 89.32,58.75 89.71,58.66 90.11,58.59 90.5,58.55 90.9,58.54 91.3,58.53 91.69,58.51 92.09,58.47 92.49,58.41 92.88,58.34 93.28,58.29 93.68,58.25 94.07,58.22 94.47,58.18 94.87,58.11 95.26,58 95.66,57.88 96.06,57.76 96.45,57.68 96.85,57.65 97.25,57.65 97.64,57.67 98.04,57.66 98.43,57.63 98.83,57.57 99.23,57.52 99.62,57.48 100.02,57.45 100.42,57.44 100.81,57.43 101.21,57.43 101.61,57.44 102,57.43 102.4,57.39 102.8,57.33 103.19,57.27 103.59,57.22 103.99,57.19 104.38,57.18 104.78,57.15 105.18,57.11 105.57,57.06 105.97,57.01 106.36,56.97 106.76,56.94 107.16,56.91 107.55,56.86 107.95,56.81 108.35,56.77 108.74,56.73 109.14,56.72 109.54,56.74 109.93,56.77 110.33,56.79 110.73,56.79 111.12,56.78 111.52,56.76 111.92,56.73 112.31,56.67 112.71,56.59 113.11,56.5 113.5,56.42 113.9,56.38 114.29,56.39 114.69,56.44 115.09,56.49 115.48,56.52 115.88,56.51 116.28,56.46 116.67,56.39 117.07,56.33 117.47,56.3 117.86,56.31 118.26,56.32 118.66,56.33 119.05,56.32 119.45,56.31 119.85,56.3 120.24,56.3 120.64,56.29 121.04,56.28 121.43,56.28 121.83,56.29 122.22,56.31 122.62,56.34 123.02,56.35 123.41,56.3 123.81,56.22 124.21,56.14 124.6,56.09 125,56.07 125.4,56.08 125.79,56.11 126.19,56.14 126.59,56.16 126.98,56.17 127.38,56.15 127.78,56.1 128.17,56.04 128.57,56 128.97,55.99 129.36,56.01 129.76,56.04 130.15,56.04 130.55,56.02 130.95,55.99 131.34,55.95 131.74,55.91 132.14,55.87 132.53,55.86 132.93,55.87 133.33,55.9 133.72,55.91 134.12,55.92 134.52,55.91 134.91,55.91 135.31,55.91 135.71,55.91 136.1,55.9 136.5,55.87 136.9,55.84 137.29,55.81 137.69,55.79 138.08,55.79 138.48,55.79 138.88,55.79 139.27,55.79 139.67,55.79 140.07,55.8 140.46,55.8 140.86,55.78 141.26,55.75 141.65,55.73 142.05,55.73 142.45,55.75 142.84,55.79 143.24,55.82 143.64,55.83 144.03,55.82 144.43,55.81 144.82,55.8 145.22,55.79 145.62,55.79 146.01,55.79 146.41,55.8 146.81,55.83 147.2,55.86 147.6,55.86 148,55.83 148.39,55.78 148.79,55.72 149.19,55.67 149.58,55.63 149.98,55.61 150.38,55.62 150.77,55.63 151.17,55.66 151.57,55.68 151.96,55.71 152.36,55.73 152.75,55.73 153.15,55.7 153.55,55.66 153.94,55.62 154.34,55.59 154.74,55.57 155.13,55.55 155.53,55.54 155.93,55.54 156.32,55.55 156.72,55.56 157.12,55.56 157.51,55.55 157.91,55.55 158.31,55.56 158.7,55.58 159.1,55.58 159.5,55.57 159.89,55.54 160.29,55.51 160.68,55.47 161.08,55.45 161.48,55.45 161.87,55.48 162.27,55.53 162.67,55.58 163.06,55.63 163.46,55.66 163.86,55.66 164.25,55.64 164.65,55.6 165.05,55.56 165.44,55.51 165.84,55.47 166.24,55.45 166.63,55.45 167.03,55.45 167.43,55.44 167.82,55.44 168.22,55.44 168.61,55.46 169.01,55.49 169.41,55.51 169.8,55.54 170.2,55.56 170.6,55.59 170.99,55.59 171.39,55.56 171.79,55.52 172.18,55.48 172.58,55.45 172.98,55.44 173.37,55.44 173.77,55.43 174.17,55.41 174.56,55.39 174.96,55.38 175.36,55.41 175.75,55.44 176.15,55.47 176.54,55.47 176.94,55.46 177.34,55.46 177.73,55.47 178.13,55.48 178.53,55.48 178.92,55.47 179.32,55.45 179.72,55.43 180.11,55.42 180.51,55.41 180.91,55.41 181.3,55.41 181.7,55.39 182.1,55.37 182.49,55.35 182.89,55.35 183.29,55.37 183.68,55.39 184.08,55.4 184.47,55.4 184.87,55.38 185.27,55.37 185.66,55.36 186.06,55.34 186.46,55.34 186.85,55.35 187.25,55.36 187.65,55.37 188.04,55.37 188.44,55.35 188.84,55.32 189.23,55.29 189.63,55.28 190.03,55.29 190.42,55.3 190.82,55.32 191.22,55.33 191.61,55.33 192.01,55.33 192.4,55.33 192.8,55.34 193.2,55.34 193.59,55.34 193.99,55.34 194.39,55.35 194.78,55.38 195.18,55.39 195.58,55.39 195.97,55.36 196.37,55.33 196.77,55.29 197.16,55.28 197.56,55.28 197.96,55.3 198.35,55.31 198.75,55.31 199.15,55.3 199.54,55.29 199.94,55.29 200.33,55.3 200.73,55.3 201.13,55.3 201.52,55.29 201.92,55.28 202.32,55.27 202.71,55.27 203.11,55.29 203.51,55.3 203.9,55.31 204.3,55.3 204.7,55.3 205.09,55.29 205.49,55.29 205.89,55.29 206.28,55.28 206.68,55.27 207.08,55.26 207.47,55.25 207.87,55.24 208.26,55.24 208.66,55.23 209.06,55.23 209.45,55.23 209.85,55.23 210.25,55.24 210.64,55.27 211.04,55.28 211.44,55.29 211.83,55.3 212.23,55.31 212.63,55.32 213.02,55.32 213.42,55.32 213.82,55.31 214.21,55.3 214.61,55.3 215.01,55.3 215.4,55.31 215.8,55.31 216.19,55.29 216.59,55.27 216.99,55.25 217.38,55.22 217.78,55.21 218.18,55.2 218.57,55.2 218.97,55.2 219.37,55.22 219.76,55.25 220.16,55.28 220.56,55.3 220.95,55.29 221.35,55.26 221.75,55.23 222.14,55.21 222.54,55.2 222.94,55.2 223.33,55.21 223.73,55.22 224.12,55.23 224.52,55.22 224.92,55.22 225.31,55.22 225.71,55.22 226.11,55.24 226.5,55.26 226.9,55.29 227.3,55.31 227.69,55.3 228.09,55.27 228.49,55.24 228.88,55.22 229.28,55.21 229.68,55.21 230.07,55.21 230.47,55.2 230.87,55.19 231.26,55.18 231.66,55.17 232.05,55.17 232.45,55.17 232.85,55.18 233.24,55.18 233.64,55.19 234.04,55.21 234.43,55.24 234.83,55.27 235.23,55.28 235.62,55.27 236.02,55.26 236.42,55.24 236.81,55.24 237.21,55.24 237.61,55.24 238,55.23 238.4,55.22 238.8,55.19 239.19,55.18 239.59,55.18 239.98,55.18 240.38,55.19 240.78,55.18 241.17,55.18 241.57,55.17 241.97,55.16 242.36,55.16 242.76,55.16 243.16,55.16 243.55,55.16 243.95,55.16 244.35,55.15 244.74,55.15 245.14,55.14 245.54,55.14 245.93,55.13 246.33,55.14 246.72,55.15 247.12,55.17 247.52,55.18 247.91,55.18 248.31,55.18 248.71,55.17 249.1,55.17 249.5,55.18 249.9,55.18 250.29,55.17 250.69,55.16 251.09,55.15 251.48,55.15 251.88,55.15 252.28,55.17 252.67,55.18 253.07,55.2 253.47,55.21 253.86,55.21 254.26,55.2 254.65,55.18 255.05,55.17 255.45,55.16 255.84,55.16 256.24,55.16 256.64,55.17 257.03,55.16 257.43,55.16 257.83,55.14 258.22,55.13 258.62,55.12 259.02,55.11 259.41,55.09 259.81,55.08 260.21,55.06 260.6,55.05 261,55.04 261.4,55.04 261.79,55.04" stroke-dasharray="none" stroke="rgb(228,26,28)" stroke-width="0.75" opacity="1" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.xyplot.lines.group.2.panel.1.1.1">
            <polyline id="Fig:densplot_01.xyplot.lines.group.2.panel.1.1.1.1" points="59.13,55.09 59.52,55.38 59.92,56.73 60.32,61.45 60.71,73.91 61.11,98.76 61.51,136.25 61.9,178.71 62.3,213.47 62.7,230.75 63.09,229.01 63.49,213.49 63.88,191.31 64.28,167.99 64.68,146.69 65.07,128.81 65.47,114.65 65.87,103.84 66.26,95.77 66.66,89.75 67.06,85.19 67.45,81.66 67.85,78.89 68.25,76.7 68.64,74.89 69.04,73.35 69.44,72.02 69.83,70.85 70.23,69.81 70.62,68.85 71.02,67.99 71.42,67.22 71.81,66.56 72.21,66.02 72.61,65.57 73,65.18 73.4,64.83 73.8,64.51 74.19,64.23 74.59,63.95 74.99,63.65 75.38,63.33 75.78,63.04 76.18,62.81 76.57,62.62 76.97,62.44 77.37,62.23 77.76,62 78.16,61.78 78.55,61.58 78.95,61.41 79.35,61.29 79.74,61.24 80.14,61.24 80.54,61.23 80.93,61.16 81.33,61 81.73,60.76 82.12,60.5 82.52,60.27 82.92,60.09 83.31,59.96 83.71,59.86 84.11,59.77 84.5,59.7 84.9,59.63 85.29,59.51 85.69,59.39 86.09,59.32 86.48,59.32 86.88,59.32 87.28,59.27 87.67,59.15 88.07,59.02 88.47,58.9 88.86,58.83 89.26,58.79 89.66,58.74 90.05,58.66 90.45,58.57 90.85,58.5 91.24,58.43 91.64,58.36 92.04,58.28 92.43,58.22 92.83,58.19 93.22,58.19 93.62,58.18 94.02,58.15 94.41,58.1 94.81,58.04 95.21,57.98 95.6,57.94 96,57.93 96.4,57.94 96.79,57.93 97.19,57.9 97.59,57.84 97.98,57.76 98.38,57.65 98.78,57.53 99.17,57.42 99.57,57.34 99.96,57.31 100.36,57.3 100.76,57.3 101.15,57.34 101.55,57.4 101.95,57.46 102.34,57.47 102.74,57.43 103.14,57.37 103.53,57.33 103.93,57.29 104.33,57.24 104.72,57.18 105.12,57.13 105.52,57.07 105.91,57.02 106.31,56.97 106.71,56.94 107.1,56.9 107.5,56.86 107.89,56.82 108.29,56.76 108.69,56.7 109.08,56.65 109.48,56.63 109.88,56.61 110.27,56.59 110.67,56.54 111.07,56.5 111.46,56.46 111.86,56.46 112.26,56.52 112.65,56.61 113.05,56.68 113.45,56.68 113.84,56.63 114.24,56.58 114.63,56.54 115.03,56.49 115.43,56.44 115.82,56.39 116.22,56.34 116.62,56.27 117.01,56.2 117.41,56.17 117.81,56.17 118.2,56.2 118.6,56.26 119,56.32 119.39,56.34 119.79,56.31 120.19,56.23 120.58,56.16 120.98,56.14 121.38,56.16 121.77,56.2 122.17,56.24 122.56,56.27 122.96,56.29 123.36,56.31 123.75,56.31 124.15,56.3 124.55,56.27 124.94,56.23 125.34,56.19 125.74,56.15 126.13,56.12 126.53,56.1 126.93,56.09 127.32,56.1 127.72,56.1 128.12,56.08 128.51,56.05 128.91,56.05 129.3,56.06 129.7,56.08 130.1,56.09 130.49,56.08 130.89,56.06 131.29,56.06 131.68,56.07 132.08,56.08 132.48,56.09 132.87,56.09 133.27,56.06 133.67,56 134.06,55.92 134.46,55.84 134.86,55.81 135.25,55.84 135.65,55.89 136.05,55.93 136.44,55.95 136.84,55.92 137.23,55.88 137.63,55.86 138.03,55.86 138.42,55.88 138.82,55.91 139.22,55.93 139.61,55.93 140.01,55.91 140.41,55.86 140.8,55.8 141.2,55.76 141.6,55.76 141.99,55.78 142.39,55.81 142.79,55.81 143.18,55.79 143.58,55.74 143.97,55.71 144.37,55.71 144.77,55.75 145.16,55.8 145.56,55.83 145.96,55.82 146.35,55.79 146.75,55.75 147.15,55.7 147.54,55.66 147.94,55.64 148.34,55.64 148.73,55.65 149.13,55.65 149.53,55.65 149.92,55.64 150.32,55.62 150.72,55.6 151.11,55.59 151.51,55.59 151.9,55.6 152.3,55.61 152.7,55.61 153.09,55.61 153.49,55.61 153.89,55.6 154.28,55.57 154.68,55.54 155.08,55.51 155.47,55.49 155.87,55.49 156.27,55.49 156.66,55.5 157.06,55.51 157.46,55.52 157.85,55.5 158.25,55.47 158.64,55.44 159.04,55.43 159.44,55.44 159.83,55.46 160.23,55.5 160.63,55.54 161.02,55.57 161.42,55.58 161.82,55.57 162.21,55.56 162.61,55.56 163.01,55.57 163.4,55.59 163.8,55.6 164.2,55.58 164.59,55.53 164.99,55.5 165.39,55.5 165.78,55.52 166.18,55.53 166.57,55.52 166.97,55.51 167.37,55.48 167.76,55.46 168.16,55.45 168.56,55.46 168.95,55.47 169.35,55.49 169.75,55.49 170.14,55.47 170.54,55.44 170.94,55.42 171.33,55.42 171.73,55.45 172.13,55.48 172.52,55.5 172.92,55.5 173.32,55.48 173.71,55.46 174.11,55.46 174.5,55.47 174.9,55.49 175.3,55.49 175.69,55.48 176.09,55.47 176.49,55.47 176.88,55.48 177.28,55.47 177.68,55.44 178.07,55.41 178.47,55.41 178.87,55.42 179.26,55.43 179.66,55.43 180.06,55.42 180.45,55.4 180.85,55.39 181.24,55.38 181.64,55.38 182.04,55.4 182.43,55.41 182.83,55.41 183.23,55.39 183.62,55.36 184.02,55.34 184.42,55.34 184.81,55.35 185.21,55.36 185.61,55.34 186,55.32 186.4,55.3 186.8,55.31 187.19,55.34 187.59,55.38 187.99,55.42 188.38,55.44 188.78,55.44 189.17,55.44 189.57,55.42 189.97,55.4 190.36,55.39 190.76,55.38 191.16,55.39 191.55,55.41 191.95,55.41 192.35,55.39 192.74,55.36 193.14,55.31 193.54,55.27 193.93,55.26 194.33,55.28 194.73,55.33 195.12,55.37 195.52,55.37 195.91,55.36 196.31,55.34 196.71,55.34 197.1,55.34 197.5,55.34 197.9,55.34 198.29,55.34 198.69,55.34 199.09,55.34 199.48,55.34 199.88,55.33 200.28,55.3 200.67,55.26 201.07,55.23 201.47,55.21 201.86,55.2 202.26,55.21 202.66,55.22 203.05,55.23 203.45,55.25 203.84,55.26 204.24,55.27 204.64,55.25 205.03,55.24 205.43,55.22 205.83,55.21 206.22,55.21 206.62,55.22 207.02,55.22 207.41,55.23 207.81,55.24 208.21,55.26 208.6,55.27 209,55.3 209.4,55.31 209.79,55.32 210.19,55.32 210.58,55.31 210.98,55.29 211.38,55.28 211.77,55.26 212.17,55.25 212.57,55.25 212.96,55.26 213.36,55.26 213.76,55.25 214.15,55.23 214.55,55.22 214.95,55.22 215.34,55.22 215.74,55.24 216.14,55.25 216.53,55.27 216.93,55.27 217.33,55.27 217.72,55.27 218.12,55.27 218.51,55.27 218.91,55.28 219.31,55.28 219.7,55.27 220.1,55.25 220.5,55.21 220.89,55.18 221.29,55.15 221.69,55.15 222.08,55.16 222.48,55.18 222.88,55.21 223.27,55.24 223.67,55.26 224.07,55.27 224.46,55.28 224.86,55.27 225.25,55.26 225.65,55.25 226.05,55.24 226.44,55.24 226.84,55.23 227.24,55.22 227.63,55.21 228.03,55.21 228.43,55.22 228.82,55.24 229.22,55.25 229.62,55.27 230.01,55.28 230.41,55.27 230.81,55.25 231.2,55.22 231.6,55.19 232,55.17 232.39,55.17 232.79,55.18 233.18,55.18 233.58,55.18 233.98,55.18 234.37,55.18 234.77,55.19 235.17,55.21 235.56,55.22 235.96,55.23 236.36,55.23 236.75,55.23 237.15,55.24 237.55,55.24 237.94,55.24 238.34,55.22 238.74,55.19 239.13,55.15 239.53,55.13 239.92,55.12 240.32,55.13 240.72,55.16 241.11,55.19 241.51,55.21 241.91,55.23 242.3,55.24 242.7,55.25 243.1,55.26 243.49,55.25 243.89,55.24 244.29,55.21 244.68,55.19 245.08,55.18 245.48,55.17 245.87,55.16 246.27,55.15 246.67,55.16 247.06,55.18 247.46,55.21 247.85,55.23 248.25,55.25 248.65,55.24 249.04,55.22 249.44,55.2 249.84,55.19 250.23,55.18 250.63,55.18 251.03,55.18 251.42,55.19 251.82,55.19 252.22,55.17 252.61,55.15 253.01,55.13 253.41,55.12 253.8,55.12 254.2,55.12 254.59,55.11 254.99,55.11 255.39,55.1 255.78,55.1 256.18,55.11 256.58,55.12 256.97,55.13 257.37,55.14 257.77,55.15 258.16,55.15 258.56,55.14 258.96,55.12 259.35,55.1 259.75,55.08 260.15,55.06 260.54,55.05 260.94,55.04 261.34,55.04 261.73,55.04" stroke-dasharray="none" stroke="rgb(55,126,184)" stroke-width="0.75" opacity="1" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.xyplot.lines.group.3.panel.1.1.1">
            <polyline id="Fig:densplot_01.xyplot.lines.group.3.panel.1.1.1.1" points="59.02,55.07 59.42,55.24 59.82,56.14 60.21,59.54 60.61,69.29 61.01,90.45 61.4,125.26 61.8,168.4 62.2,207.57 62.59,230.83 62.99,233.75 63.39,220.1 63.78,197.44 64.18,172.67 64.58,150.01 64.97,131.18 65.37,116.32 65.77,104.92 66.16,96.33 66.56,89.93 66.96,85.15 67.35,81.53 67.75,78.78 68.15,76.66 68.55,74.96 68.94,73.53 69.34,72.3 69.74,71.23 70.13,70.27 70.53,69.38 70.93,68.51 71.32,67.69 71.72,66.97 72.12,66.39 72.51,65.95 72.91,65.59 73.31,65.22 73.7,64.77 74.1,64.29 74.5,63.82 74.89,63.44 75.29,63.17 75.69,63.01 76.09,62.87 76.48,62.69 76.88,62.44 77.28,62.21 77.67,62.06 78.07,61.98 78.47,61.91 78.86,61.8 79.26,61.63 79.66,61.43 80.05,61.25 80.45,61.09 80.85,60.95 81.24,60.82 81.64,60.71 82.04,60.61 82.43,60.53 82.83,60.44 83.23,60.32 83.63,60.16 84.02,59.96 84.42,59.76 84.82,59.62 85.21,59.54 85.61,59.49 86.01,59.4 86.4,59.29 86.8,59.15 87.2,59.03 87.59,58.95 87.99,58.91 88.39,58.9 88.78,58.88 89.18,58.84 89.58,58.79 89.97,58.73 90.37,58.65 90.77,58.55 91.16,58.46 91.56,58.42 91.96,58.43 92.36,58.44 92.75,58.41 93.15,58.34 93.55,58.25 93.94,58.18 94.34,58.16 94.74,58.17 95.13,58.16 95.53,58.08 95.93,57.95 96.32,57.8 96.72,57.69 97.12,57.66 97.51,57.69 97.91,57.74 98.31,57.75 98.7,57.71 99.1,57.61 99.5,57.5 99.9,57.42 100.29,57.37 100.69,57.35 101.09,57.33 101.48,57.33 101.88,57.34 102.28,57.34 102.67,57.31 103.07,57.26 103.47,57.18 103.86,57.1 104.26,57.05 104.66,57.03 105.05,57.03 105.45,57.04 105.85,57.04 106.24,57.03 106.64,57 107.04,56.94 107.44,56.85 107.83,56.75 108.23,56.68 108.63,56.68 109.02,56.73 109.42,56.79 109.82,56.82 110.21,56.79 110.61,56.72 111.01,56.62 111.4,56.53 111.8,56.48 112.2,56.48 112.59,56.53 112.99,56.57 113.39,56.58 113.78,56.54 114.18,56.5 114.58,56.49 114.98,56.49 115.37,56.51 115.77,56.52 116.17,56.5 116.56,56.46 116.96,56.37 117.36,56.28 117.75,56.19 118.15,56.14 118.55,56.14 118.94,56.16 119.34,56.19 119.74,56.21 120.13,56.22 120.53,56.23 120.93,56.23 121.32,56.21 121.72,56.19 122.12,56.17 122.51,56.17 122.91,56.2 123.31,56.24 123.71,56.27 124.1,56.27 124.5,56.25 124.9,56.23 125.29,56.22 125.69,56.21 126.09,56.19 126.48,56.16 126.88,56.12 127.28,56.1 127.67,56.08 128.07,56.09 128.47,56.11 128.86,56.11 129.26,56.08 129.66,56.03 130.05,55.98 130.45,55.97 130.85,55.99 131.25,56.04 131.64,56.1 132.04,56.16 132.44,56.2 132.83,56.21 133.23,56.18 133.63,56.12 134.02,56.04 134.42,55.96 134.82,55.89 135.21,55.84 135.61,55.8 136.01,55.79 136.4,55.8 136.8,55.83 137.2,55.86 137.59,55.86 137.99,55.84 138.39,55.81 138.79,55.78 139.18,55.76 139.58,55.76 139.98,55.76 140.37,55.77 140.77,55.77 141.17,55.77 141.56,55.78 141.96,55.8 142.36,55.82 142.75,55.83 143.15,55.82 143.55,55.79 143.94,55.76 144.34,55.74 144.74,55.74 145.13,55.75 145.53,55.76 145.93,55.76 146.32,55.76 146.72,55.74 147.12,55.71 147.52,55.7 147.91,55.69 148.31,55.69 148.71,55.7 149.1,55.72 149.5,55.73 149.9,55.72 150.29,55.69 150.69,55.65 151.09,55.63 151.48,55.65 151.88,55.68 152.28,55.72 152.67,55.73 153.07,55.73 153.47,55.72 153.86,55.7 154.26,55.68 154.66,55.65 155.06,55.63 155.45,55.61 155.85,55.6 156.25,55.59 156.64,55.6 157.04,55.6 157.44,55.6 157.83,55.58 158.23,55.55 158.63,55.54 159.02,55.53 159.42,55.51 159.82,55.49 160.21,55.46 160.61,55.44 161.01,55.42 161.4,55.41 161.8,55.41 162.2,55.43 162.6,55.46 162.99,55.5 163.39,55.52 163.79,55.52 164.18,55.52 164.58,55.51 164.98,55.5 165.37,55.5 165.77,55.52 166.17,55.54 166.56,55.55 166.96,55.52 167.36,55.47 167.75,55.44 168.15,55.44 168.55,55.46 168.94,55.47 169.34,55.47 169.74,55.46 170.14,55.46 170.53,55.49 170.93,55.53 171.33,55.56 171.72,55.57 172.12,55.57 172.52,55.54 172.91,55.51 173.31,55.49 173.71,55.49 174.1,55.49 174.5,55.5 174.9,55.49 175.29,55.48 175.69,55.48 176.09,55.49 176.48,55.52 176.88,55.56 177.28,55.57 177.67,55.56 178.07,55.53 178.47,55.5 178.87,55.45 179.26,55.42 179.66,55.39 180.06,55.37 180.45,55.36 180.85,55.36 181.25,55.36 181.64,55.36 182.04,55.36 182.44,55.35 182.83,55.34 183.23,55.34 183.63,55.36 184.02,55.37 184.42,55.38 184.82,55.38 185.21,55.36 185.61,55.34 186.01,55.32 186.41,55.32 186.8,55.34 187.2,55.36 187.6,55.37 187.99,55.38 188.39,55.37 188.79,55.36 189.18,55.35 189.58,55.32 189.98,55.3 190.37,55.26 190.77,55.24 191.17,55.23 191.56,55.25 191.96,55.28 192.36,55.29 192.75,55.29 193.15,55.29 193.55,55.28 193.95,55.28 194.34,55.28 194.74,55.28 195.14,55.28 195.53,55.28 195.93,55.28 196.33,55.29 196.72,55.31 197.12,55.33 197.52,55.34 197.91,55.33 198.31,55.3 198.71,55.29 199.1,55.29 199.5,55.31 199.9,55.33 200.29,55.36 200.69,55.37 201.09,55.37 201.48,55.35 201.88,55.34 202.28,55.33 202.68,55.33 203.07,55.35 203.47,55.37 203.87,55.4 204.26,55.43 204.66,55.44 205.06,55.43 205.45,55.4 205.85,55.37 206.25,55.34 206.64,55.32 207.04,55.29 207.44,55.27 207.83,55.25 208.23,55.23 208.63,55.22 209.02,55.22 209.42,55.24 209.82,55.26 210.22,55.28 210.61,55.29 211.01,55.3 211.41,55.3 211.8,55.3 212.2,55.3 212.6,55.3 212.99,55.3 213.39,55.3 213.79,55.29 214.18,55.29 214.58,55.28 214.98,55.26 215.37,55.24 215.77,55.22 216.17,55.22 216.56,55.22 216.96,55.23 217.36,55.24 217.76,55.25 218.15,55.27 218.55,55.27 218.95,55.27 219.34,55.25 219.74,55.23 220.14,55.23 220.53,55.23 220.93,55.24 221.33,55.24 221.72,55.24 222.12,55.24 222.52,55.24 222.91,55.25 223.31,55.26 223.71,55.26 224.1,55.26 224.5,55.24 224.9,55.23 225.29,55.21 225.69,55.2 226.09,55.2 226.49,55.2 226.88,55.2 227.28,55.22 227.68,55.22 228.07,55.22 228.47,55.21 228.87,55.2 229.26,55.2 229.66,55.21 230.06,55.23 230.45,55.25 230.85,55.26 231.25,55.25 231.64,55.22 232.04,55.19 232.44,55.17 232.83,55.15 233.23,55.14 233.63,55.15 234.03,55.17 234.42,55.18 234.82,55.18 235.22,55.18 235.61,55.17 236.01,55.18 236.41,55.19 236.8,55.2 237.2,55.21 237.6,55.2 237.99,55.19 238.39,55.16 238.79,55.15 239.18,55.15 239.58,55.15 239.98,55.17 240.37,55.19 240.77,55.21 241.17,55.24 241.57,55.25 241.96,55.25 242.36,55.23 242.76,55.21 243.15,55.2 243.55,55.18 243.95,55.18 244.34,55.17 244.74,55.17 245.14,55.17 245.53,55.17 245.93,55.18 246.33,55.19 246.72,55.2 247.12,55.2 247.52,55.2 247.91,55.2 248.31,55.19 248.71,55.17 249.11,55.16 249.5,55.15 249.9,55.14 250.3,55.14 250.69,55.14 251.09,55.14 251.49,55.15 251.88,55.16 252.28,55.16 252.68,55.16 253.07,55.16 253.47,55.15 253.87,55.14 254.26,55.14 254.66,55.14 255.06,55.14 255.45,55.14 255.85,55.13 256.25,55.12 256.64,55.11 257.04,55.11 257.44,55.12 257.84,55.13 258.23,55.14 258.63,55.15 259.03,55.15 259.42,55.14 259.82,55.11 260.22,55.08 260.61,55.06 261.01,55.04 261.41,55.04 261.8,55.04" stroke-dasharray="none" stroke="rgb(77,175,74)" stroke-width="0.75" opacity="1" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.xyplot.lines.group.4.panel.1.1.1">
            <polyline id="Fig:densplot_01.xyplot.lines.group.4.panel.1.1.1.1" points="59.15,55.1 59.55,55.41 59.95,56.85 60.34,61.86 60.74,75.05 61.14,101.3 61.53,140.78 61.93,185.05 62.32,220.25 62.72,235.93 63.12,230.92 63.51,211.68 63.91,186.66 64.31,162.1 64.7,140.94 65.1,123.86 65.5,110.5 65.89,100.31 66.29,92.73 66.69,87.24 67.08,83.32 67.48,80.51 67.88,78.41 68.27,76.69 68.67,75.17 69.07,73.83 69.46,72.63 69.86,71.55 70.26,70.53 70.65,69.55 71.05,68.62 71.45,67.81 71.84,67.13 72.24,66.54 72.64,65.99 73.03,65.45 73.43,64.96 73.82,64.55 74.22,64.21 74.62,63.92 75.01,63.67 75.41,63.47 75.81,63.28 76.2,63.09 76.6,62.88 77,62.69 77.39,62.5 77.79,62.32 78.19,62.15 78.58,61.97 78.98,61.77 79.38,61.55 79.77,61.29 80.17,61.01 80.57,60.75 80.96,60.54 81.36,60.42 81.76,60.34 82.15,60.27 82.55,60.2 82.95,60.14 83.34,60.1 83.74,60.03 84.13,59.91 84.53,59.72 84.93,59.53 85.32,59.4 85.72,59.35 86.12,59.36 86.51,59.39 86.91,59.39 87.31,59.37 87.7,59.32 88.1,59.23 88.5,59.1 88.89,58.94 89.29,58.79 89.69,58.68 90.08,58.63 90.48,58.63 90.88,58.67 91.27,58.72 91.67,58.75 92.07,58.73 92.46,58.68 92.86,58.62 93.26,58.56 93.65,58.48 94.05,58.39 94.44,58.28 94.84,58.19 95.24,58.12 95.63,58.07 96.03,58.01 96.43,57.93 96.82,57.82 97.22,57.67 97.62,57.52 98.01,57.43 98.41,57.4 98.81,57.43 99.2,57.47 99.6,57.49 100,57.48 100.39,57.45 100.79,57.39 101.19,57.29 101.58,57.16 101.98,57.02 102.38,56.92 102.77,56.89 103.17,56.94 103.57,57.05 103.96,57.16 104.36,57.22 104.75,57.21 105.15,57.15 105.55,57.08 105.94,57.02 106.34,56.98 106.74,56.92 107.13,56.85 107.53,56.78 107.93,56.73 108.32,56.72 108.72,56.74 109.12,56.76 109.51,56.76 109.91,56.74 110.31,56.71 110.7,56.68 111.1,56.67 111.5,56.65 111.89,56.62 112.29,56.57 112.69,56.53 113.08,56.51 113.48,56.51 113.88,56.5 114.27,56.46 114.67,56.42 115.07,56.41 115.46,56.44 115.86,56.51 116.25,56.56 116.65,56.58 117.05,56.55 117.44,56.46 117.84,56.37 118.24,56.31 118.63,56.3 119.03,56.32 119.43,56.33 119.82,56.32 120.22,56.3 120.62,56.28 121.01,56.28 121.41,56.28 121.81,56.29 122.2,56.28 122.6,56.25 123,56.2 123.39,56.16 123.79,56.13 124.19,56.13 124.58,56.15 124.98,56.19 125.38,56.22 125.77,56.23 126.17,56.21 126.56,56.17 126.96,56.12 127.36,56.07 127.75,56.05 128.15,56.05 128.55,56.05 128.94,56.06 129.34,56.07 129.74,56.09 130.13,56.12 130.53,56.15 130.93,56.17 131.32,56.15 131.72,56.1 132.12,56.02 132.51,55.95 132.91,55.9 133.31,55.87 133.7,55.86 134.1,55.86 134.5,55.86 134.89,55.84 135.29,55.82 135.69,55.8 136.08,55.79 136.48,55.78 136.87,55.78 137.27,55.79 137.67,55.82 138.06,55.87 138.46,55.91 138.86,55.93 139.25,55.93 139.65,55.9 140.05,55.88 140.44,55.87 140.84,55.86 141.24,55.84 141.63,55.82 142.03,55.8 142.43,55.79 142.82,55.79 143.22,55.79 143.62,55.8 144.01,55.81 144.41,55.81 144.81,55.79 145.2,55.77 145.6,55.74 146,55.72 146.39,55.72 146.79,55.73 147.18,55.74 147.58,55.75 147.98,55.75 148.37,55.75 148.77,55.73 149.17,55.71 149.56,55.69 149.96,55.67 150.36,55.67 150.75,55.68 151.15,55.68 151.55,55.69 151.94,55.68 152.34,55.65 152.74,55.61 153.13,55.59 153.53,55.58 153.93,55.59 154.32,55.59 154.72,55.59 155.12,55.59 155.51,55.58 155.91,55.57 156.31,55.55 156.7,55.52 157.1,55.5 157.49,55.5 157.89,55.51 158.29,55.53 158.68,55.54 159.08,55.55 159.48,55.55 159.87,55.54 160.27,55.52 160.67,55.5 161.06,55.5 161.46,55.51 161.86,55.51 162.25,55.52 162.65,55.52 163.05,55.53 163.44,55.54 163.84,55.54 164.24,55.53 164.63,55.53 165.03,55.54 165.43,55.55 165.82,55.54 166.22,55.52 166.62,55.48 167.01,55.44 167.41,55.41 167.81,55.4 168.2,55.41 168.6,55.44 168.99,55.49 169.39,55.54 169.79,55.56 170.18,55.55 170.58,55.51 170.98,55.47 171.37,55.44 171.77,55.42 172.17,55.42 172.56,55.42 172.96,55.43 173.36,55.44 173.75,55.44 174.15,55.43 174.55,55.41 174.94,55.4 175.34,55.4 175.74,55.4 176.13,55.4 176.53,55.39 176.93,55.38 177.32,55.36 177.72,55.35 178.12,55.34 178.51,55.35 178.91,55.37 179.3,55.41 179.7,55.45 180.1,55.47 180.49,55.48 180.89,55.46 181.29,55.44 181.68,55.41 182.08,55.4 182.48,55.4 182.87,55.4 183.27,55.41 183.67,55.41 184.06,55.4 184.46,55.4 184.86,55.39 185.25,55.37 185.65,55.35 186.05,55.33 186.44,55.33 186.84,55.34 187.24,55.36 187.63,55.36 188.03,55.36 188.43,55.35 188.82,55.35 189.22,55.36 189.61,55.38 190.01,55.4 190.41,55.4 190.8,55.39 191.2,55.38 191.6,55.37 191.99,55.36 192.39,55.36 192.79,55.36 193.18,55.35 193.58,55.34 193.98,55.32 194.37,55.31 194.77,55.31 195.17,55.31 195.56,55.32 195.96,55.34 196.36,55.35 196.75,55.36 197.15,55.36 197.55,55.35 197.94,55.34 198.34,55.34 198.74,55.36 199.13,55.36 199.53,55.36 199.92,55.34 200.32,55.32 200.72,55.31 201.11,55.31 201.51,55.3 201.91,55.29 202.3,55.29 202.7,55.29 203.1,55.31 203.49,55.32 203.89,55.32 204.29,55.31 204.68,55.29 205.08,55.27 205.48,55.28 205.87,55.3 206.27,55.32 206.67,55.32 207.06,55.3 207.46,55.28 207.86,55.26 208.25,55.27 208.65,55.28 209.05,55.29 209.44,55.3 209.84,55.29 210.23,55.28 210.63,55.28 211.03,55.28 211.42,55.28 211.82,55.27 212.22,55.25 212.61,55.23 213.01,55.21 213.41,55.19 213.8,55.18 214.2,55.19 214.6,55.22 214.99,55.25 215.39,55.28 215.79,55.28 216.18,55.28 216.58,55.28 216.98,55.28 217.37,55.27 217.77,55.26 218.17,55.25 218.56,55.24 218.96,55.23 219.36,55.22 219.75,55.22 220.15,55.22 220.55,55.22 220.94,55.21 221.34,55.22 221.73,55.23 222.13,55.24 222.53,55.24 222.92,55.24 223.32,55.22 223.72,55.2 224.11,55.18 224.51,55.17 224.91,55.17 225.3,55.17 225.7,55.18 226.1,55.19 226.49,55.18 226.89,55.18 227.29,55.19 227.68,55.21 228.08,55.24 228.48,55.27 228.87,55.29 229.27,55.29 229.67,55.27 230.06,55.25 230.46,55.23 230.86,55.21 231.25,55.19 231.65,55.17 232.04,55.16 232.44,55.16 232.84,55.17 233.23,55.17 233.63,55.16 234.03,55.15 234.42,55.15 234.82,55.16 235.22,55.18 235.61,55.19 236.01,55.21 236.41,55.2 236.8,55.18 237.2,55.15 237.6,55.13 237.99,55.13 238.39,55.15 238.79,55.17 239.18,55.19 239.58,55.2 239.98,55.21 240.37,55.2 240.77,55.2 241.17,55.2 241.56,55.21 241.96,55.23 242.35,55.24 242.75,55.24 243.15,55.23 243.54,55.23 243.94,55.23 244.34,55.23 244.73,55.23 245.13,55.22 245.53,55.21 245.92,55.19 246.32,55.18 246.72,55.18 247.11,55.18 247.51,55.18 247.91,55.17 248.3,55.15 248.7,55.13 249.1,55.13 249.49,55.13 249.89,55.14 250.29,55.15 250.68,55.15 251.08,55.16 251.48,55.18 251.87,55.2 252.27,55.22 252.66,55.22 253.06,55.21 253.46,55.2 253.85,55.19 254.25,55.19 254.65,55.19 255.04,55.18 255.44,55.16 255.84,55.15 256.23,55.14 256.63,55.13 257.03,55.12 257.42,55.11 257.82,55.1 258.22,55.1 258.61,55.1 259.01,55.1 259.41,55.09 259.8,55.08 260.2,55.06 260.6,55.05 260.99,55.04 261.39,55.04 261.79,55.04" stroke-dasharray="none" stroke="rgb(152,78,163)" stroke-width="0.75" opacity="1" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.xyplot.lines.group.5.panel.1.1.1">
            <polyline id="Fig:densplot_01.xyplot.lines.group.5.panel.1.1.1.1" points="59,55.06 59.4,55.23 59.79,56.07 60.19,59.28 60.58,68.63 60.98,89.2 61.38,123.5 61.77,166.56 62.17,206.15 62.57,230.03 62.96,233.46 63.36,220.23 63.75,197.96 64.15,173.4 64.55,150.61 64.94,131.34 65.34,116 65.74,104.35 66.13,95.82 66.53,89.73 66.92,85.39 67.32,82.23 67.72,79.8 68.11,77.83 68.51,76.11 68.9,74.55 69.3,73.1 69.7,71.78 70.09,70.6 70.49,69.58 70.89,68.71 71.28,67.98 71.68,67.36 72.07,66.83 72.47,66.36 72.87,65.89 73.26,65.39 73.66,64.93 74.06,64.55 74.45,64.25 74.85,63.97 75.24,63.7 75.64,63.45 76.04,63.22 76.43,63 76.83,62.78 77.22,62.54 77.62,62.3 78.02,62.05 78.41,61.81 78.81,61.58 79.21,61.34 79.6,61.11 80,60.91 80.39,60.77 80.79,60.7 81.19,60.7 81.58,60.73 81.98,60.7 82.38,60.57 82.77,60.35 83.17,60.09 83.56,59.88 83.96,59.76 84.36,59.72 84.75,59.72 85.15,59.67 85.54,59.56 85.94,59.41 86.34,59.26 86.73,59.13 87.13,59.03 87.53,58.97 87.92,58.93 88.32,58.89 88.71,58.86 89.11,58.83 89.51,58.81 89.9,58.81 90.3,58.82 90.7,58.81 91.09,58.76 91.49,58.66 91.88,58.53 92.28,58.41 92.68,58.34 93.07,58.3 93.47,58.28 93.86,58.24 94.26,58.19 94.66,58.15 95.05,58.15 95.45,58.17 95.85,58.15 96.24,58.06 96.64,57.89 97.03,57.71 97.43,57.58 97.83,57.54 98.22,57.55 98.62,57.54 99.02,57.47 99.41,57.34 99.81,57.22 100.2,57.14 100.6,57.12 101,57.13 101.39,57.11 101.79,57.08 102.19,57.06 102.58,57.07 102.98,57.11 103.37,57.14 103.77,57.18 104.17,57.21 104.56,57.25 104.96,57.28 105.35,57.26 105.75,57.19 106.15,57.07 106.54,56.96 106.94,56.89 107.34,56.86 107.73,56.86 108.13,56.87 108.52,56.88 108.92,56.87 109.32,56.86 109.71,56.82 110.11,56.76 110.51,56.7 110.9,56.64 111.3,56.61 111.69,56.61 112.09,56.62 112.49,56.63 112.88,56.64 113.28,56.64 113.67,56.62 114.07,56.57 114.47,56.51 114.86,56.45 115.26,56.41 115.66,56.4 116.05,56.39 116.45,56.36 116.84,56.29 117.24,56.23 117.64,56.21 118.03,56.23 118.43,56.27 118.83,56.32 119.22,56.39 119.62,56.46 120.01,56.49 120.41,56.47 120.81,56.42 121.2,56.36 121.6,56.31 121.99,56.25 122.39,56.19 122.79,56.14 123.18,56.09 123.58,56.07 123.98,56.08 124.37,56.1 124.77,56.12 125.16,56.14 125.56,56.14 125.96,56.13 126.35,56.11 126.75,56.1 127.15,56.1 127.54,56.1 127.94,56.09 128.33,56.07 128.73,56.04 129.13,56 129.52,55.96 129.92,55.93 130.31,55.93 130.71,55.95 131.11,55.99 131.5,56.03 131.9,56.06 132.3,56.08 132.69,56.09 133.09,56.06 133.48,56 133.88,55.95 134.28,55.91 134.67,55.89 135.07,55.88 135.47,55.87 135.86,55.87 136.26,55.85 136.65,55.84 137.05,55.82 137.45,55.81 137.84,55.81 138.24,55.82 138.63,55.83 139.03,55.83 139.43,55.83 139.82,55.83 140.22,55.82 140.62,55.8 141.01,55.79 141.41,55.79 141.8,55.8 142.2,55.8 142.6,55.8 142.99,55.79 143.39,55.79 143.79,55.78 144.18,55.78 144.58,55.78 144.97,55.76 145.37,55.74 145.77,55.7 146.16,55.67 146.56,55.66 146.95,55.67 147.35,55.69 147.75,55.72 148.14,55.73 148.54,55.74 148.94,55.74 149.33,55.75 149.73,55.74 150.12,55.73 150.52,55.7 150.92,55.67 151.31,55.65 151.71,55.63 152.11,55.61 152.5,55.59 152.9,55.58 153.29,55.58 153.69,55.59 154.09,55.61 154.48,55.62 154.88,55.62 155.27,55.62 155.67,55.6 156.07,55.59 156.46,55.59 156.86,55.59 157.26,55.58 157.65,55.57 158.05,55.55 158.44,55.55 158.84,55.57 159.24,55.59 159.63,55.6 160.03,55.6 160.43,55.59 160.82,55.58 161.22,55.56 161.61,55.54 162.01,55.53 162.41,55.53 162.8,55.53 163.2,55.53 163.59,55.53 163.99,55.54 164.39,55.55 164.78,55.55 165.18,55.54 165.58,55.53 165.97,55.51 166.37,55.49 166.76,55.48 167.16,55.48 167.56,55.5 167.95,55.52 168.35,55.55 168.75,55.57 169.14,55.59 169.54,55.59 169.93,55.56 170.33,55.51 170.73,55.46 171.12,55.42 171.52,55.4 171.91,55.4 172.31,55.41 172.71,55.42 173.1,55.42 173.5,55.41 173.9,55.39 174.29,55.37 174.69,55.36 175.08,55.37 175.48,55.38 175.88,55.38 176.27,55.38 176.67,55.37 177.07,55.36 177.46,55.35 177.86,55.34 178.25,55.34 178.65,55.35 179.05,55.37 179.44,55.38 179.84,55.36 180.23,55.35 180.63,55.36 181.03,55.39 181.42,55.42 181.82,55.44 182.22,55.43 182.61,55.42 183.01,55.42 183.4,55.44 183.8,55.46 184.2,55.47 184.59,55.47 184.99,55.46 185.39,55.44 185.78,55.42 186.18,55.41 186.57,55.41 186.97,55.43 187.37,55.43 187.76,55.42 188.16,55.4 188.55,55.39 188.95,55.4 189.35,55.4 189.74,55.39 190.14,55.37 190.54,55.35 190.93,55.33 191.33,55.32 191.72,55.32 192.12,55.33 192.52,55.34 192.91,55.34 193.31,55.33 193.71,55.31 194.1,55.3 194.5,55.31 194.89,55.33 195.29,55.33 195.69,55.33 196.08,55.32 196.48,55.31 196.87,55.3 197.27,55.29 197.67,55.29 198.06,55.29 198.46,55.29 198.86,55.31 199.25,55.33 199.65,55.34 200.04,55.33 200.44,55.31 200.84,55.29 201.23,55.28 201.63,55.29 202.03,55.3 202.42,55.31 202.82,55.31 203.21,55.31 203.61,55.31 204.01,55.32 204.4,55.32 204.8,55.33 205.19,55.33 205.59,55.32 205.99,55.29 206.38,55.26 206.78,55.25 207.18,55.24 207.57,55.24 207.97,55.24 208.36,55.25 208.76,55.27 209.16,55.29 209.55,55.31 209.95,55.31 210.35,55.32 210.74,55.32 211.14,55.32 211.53,55.31 211.93,55.3 212.33,55.28 212.72,55.27 213.12,55.26 213.51,55.25 213.91,55.23 214.31,55.21 214.7,55.2 215.1,55.21 215.5,55.23 215.89,55.25 216.29,55.25 216.68,55.23 217.08,55.21 217.48,55.18 217.87,55.17 218.27,55.18 218.67,55.2 219.06,55.23 219.46,55.26 219.85,55.29 220.25,55.29 220.65,55.27 221.04,55.23 221.44,55.2 221.83,55.18 222.23,55.18 222.63,55.19 223.02,55.2 223.42,55.22 223.82,55.23 224.21,55.23 224.61,55.22 225,55.21 225.4,55.2 225.8,55.21 226.19,55.21 226.59,55.21 226.99,55.2 227.38,55.19 227.78,55.18 228.17,55.19 228.57,55.21 228.97,55.23 229.36,55.24 229.76,55.25 230.15,55.26 230.55,55.25 230.95,55.24 231.34,55.23 231.74,55.23 232.14,55.24 232.53,55.26 232.93,55.27 233.32,55.27 233.72,55.26 234.12,55.25 234.51,55.24 234.91,55.23 235.31,55.22 235.7,55.22 236.1,55.21 236.49,55.21 236.89,55.21 237.29,55.21 237.68,55.2 238.08,55.2 238.47,55.2 238.87,55.2 239.27,55.22 239.66,55.22 240.06,55.2 240.46,55.18 240.85,55.15 241.25,55.14 241.64,55.14 242.04,55.15 242.44,55.16 242.83,55.17 243.23,55.17 243.63,55.17 244.02,55.15 244.42,55.14 244.81,55.14 245.21,55.15 245.61,55.17 246,55.19 246.4,55.21 246.8,55.22 247.19,55.21 247.59,55.2 247.98,55.19 248.38,55.19 248.78,55.19 249.17,55.18 249.57,55.17 249.96,55.15 250.36,55.14 250.76,55.14 251.15,55.14 251.55,55.15 251.95,55.15 252.34,55.16 252.74,55.16 253.13,55.15 253.53,55.15 253.93,55.15 254.32,55.15 254.72,55.15 255.12,55.15 255.51,55.15 255.91,55.15 256.3,55.16 256.7,55.17 257.1,55.17 257.49,55.17 257.89,55.16 258.28,55.13 258.68,55.11 259.08,55.09 259.47,55.07 259.87,55.05 260.27,55.04 260.66,55.04 261.06,55.04 261.45,55.04" stroke-dasharray="none" stroke="rgb(255,127,0)" stroke-width="0.75" opacity="1" stroke-opacity="1" fill="none"/>
          </g>
          <g id="Fig:densplot_01.xyplot.lines.group.6.panel.1.1.1">
            <polyline id="Fig:densplot_01.xyplot.lines.group.6.panel.1.1.1.1" points="59.04,55.07 59.43,55.26 59.83,56.21 60.23,59.76 60.62,69.82 61.02,91.42 61.42,126.62 61.81,169.77 62.21,208.34 62.61,230.51 63,232.32 63.4,218.11 63.79,195.58 64.19,171.28 64.59,149.01 64.98,130.32 65.38,115.54 65.78,104.37 66.17,96.19 66.57,90.23 66.97,85.76 67.36,82.31 67.76,79.65 68.16,77.58 68.55,75.89 68.95,74.34 69.35,72.81 69.74,71.35 70.14,70.09 70.54,69.09 70.93,68.32 71.33,67.67 71.73,67.07 72.12,66.54 72.52,66.09 72.92,65.7 73.31,65.28 73.71,64.81 74.11,64.36 74.5,64.02 74.9,63.8 75.29,63.62 75.69,63.42 76.09,63.17 76.48,62.9 76.88,62.63 77.28,62.37 77.67,62.13 78.07,61.92 78.47,61.75 78.86,61.59 79.26,61.44 79.66,61.32 80.05,61.22 80.45,61.11 80.85,60.97 81.24,60.81 81.64,60.66 82.04,60.54 82.43,60.43 82.83,60.3 83.23,60.15 83.62,59.99 84.02,59.87 84.42,59.82 84.81,59.84 85.21,59.87 85.61,59.82 86,59.65 86.4,59.42 86.79,59.22 87.19,59.06 87.59,58.93 87.98,58.8 88.38,58.69 88.78,58.65 89.17,58.66 89.57,58.71 89.97,58.74 90.36,58.75 90.76,58.74 91.16,58.72 91.55,58.67 91.95,58.59 92.35,58.5 92.74,58.44 93.14,58.41 93.54,58.41 93.93,58.39 94.33,58.33 94.73,58.23 95.12,58.13 95.52,58.03 95.92,57.94 96.31,57.83 96.71,57.73 97.11,57.65 97.5,57.61 97.9,57.59 98.29,57.59 98.69,57.58 99.09,57.55 99.48,57.51 99.88,57.48 100.28,57.46 100.67,57.43 101.07,57.38 101.47,57.3 101.86,57.22 102.26,57.17 102.66,57.13 103.05,57.09 103.45,57.03 103.85,56.95 104.24,56.87 104.64,56.81 105.04,56.78 105.43,56.78 105.83,56.8 106.23,56.83 106.62,56.85 107.02,56.87 107.42,56.92 107.81,56.98 108.21,57.02 108.61,56.99 109,56.9 109.4,56.78 109.8,56.68 110.19,56.63 110.59,56.61 110.98,56.62 111.38,56.61 111.78,56.59 112.17,56.56 112.57,56.53 112.97,56.51 113.36,56.48 113.76,56.46 114.16,56.46 114.55,56.49 114.95,56.52 115.35,56.54 115.74,56.52 116.14,56.47 116.54,56.43 116.93,56.39 117.33,56.36 117.73,56.35 118.12,56.34 118.52,56.33 118.92,56.32 119.31,56.31 119.71,56.3 120.11,56.29 120.5,56.26 120.9,56.25 121.3,56.25 121.69,56.26 122.09,56.27 122.48,56.27 122.88,56.24 123.28,56.23 123.67,56.23 124.07,56.24 124.47,56.24 124.86,56.25 125.26,56.26 125.66,56.26 126.05,56.23 126.45,56.18 126.85,56.1 127.24,56.05 127.64,56.02 128.04,56.01 128.43,56.02 128.83,56.02 129.23,56.01 129.62,55.99 130.02,55.97 130.42,55.93 130.81,55.91 131.21,55.91 131.61,55.94 132,55.97 132.4,55.99 132.8,56.02 133.19,56.03 133.59,56.04 133.98,56.04 134.38,56.02 134.78,55.98 135.17,55.93 135.57,55.87 135.97,55.83 136.36,55.82 136.76,55.82 137.16,55.83 137.55,55.83 137.95,55.81 138.35,55.79 138.74,55.78 139.14,55.77 139.54,55.77 139.93,55.78 140.33,55.82 140.73,55.86 141.12,55.89 141.52,55.89 141.92,55.86 142.31,55.82 142.71,55.79 143.11,55.75 143.5,55.73 143.9,55.73 144.3,55.75 144.69,55.79 145.09,55.83 145.48,55.86 145.88,55.84 146.28,55.77 146.67,55.7 147.07,55.64 147.47,55.64 147.86,55.66 148.26,55.71 148.66,55.75 149.05,55.79 149.45,55.79 149.85,55.77 150.24,55.73 150.64,55.68 151.04,55.64 151.43,55.61 151.83,55.59 152.23,55.6 152.62,55.62 153.02,55.64 153.42,55.64 153.81,55.62 154.21,55.6 154.61,55.58 155,55.59 155.4,55.61 155.8,55.65 156.19,55.68 156.59,55.69 156.99,55.69 157.38,55.67 157.78,55.65 158.17,55.64 158.57,55.63 158.97,55.62 159.36,55.6 159.76,55.57 160.16,55.53 160.55,55.48 160.95,55.44 161.35,55.43 161.74,55.44 162.14,55.46 162.54,55.47 162.93,55.46 163.33,55.44 163.73,55.42 164.12,55.41 164.52,55.43 164.92,55.45 165.31,55.48 165.71,55.51 166.11,55.53 166.5,55.54 166.9,55.55 167.3,55.56 167.69,55.55 168.09,55.53 168.49,55.49 168.88,55.44 169.28,55.41 169.67,55.4 170.07,55.41 170.47,55.45 170.86,55.48 171.26,55.48 171.66,55.47 172.05,55.44 172.45,55.43 172.85,55.42 173.24,55.44 173.64,55.47 174.04,55.5 174.43,55.52 174.83,55.51 175.23,55.48 175.62,55.44 176.02,55.41 176.42,55.39 176.81,55.38 177.21,55.38 177.61,55.4 178,55.42 178.4,55.45 178.8,55.46 179.19,55.46 179.59,55.45 179.99,55.44 180.38,55.43 180.78,55.44 181.17,55.45 181.57,55.46 181.97,55.47 182.36,55.46 182.76,55.46 183.16,55.46 183.55,55.46 183.95,55.46 184.35,55.46 184.74,55.44 185.14,55.42 185.54,55.39 185.93,55.35 186.33,55.3 186.73,55.27 187.12,55.26 187.52,55.27 187.92,55.28 188.31,55.29 188.71,55.3 189.11,55.32 189.5,55.34 189.9,55.36 190.3,55.37 190.69,55.36 191.09,55.34 191.49,55.33 191.88,55.33 192.28,55.32 192.67,55.3 193.07,55.3 193.47,55.3 193.86,55.33 194.26,55.37 194.66,55.4 195.05,55.43 195.45,55.44 195.85,55.42 196.24,55.39 196.64,55.34 197.04,55.31 197.43,55.3 197.83,55.32 198.23,55.34 198.62,55.36 199.02,55.36 199.42,55.35 199.81,55.32 200.21,55.29 200.61,55.29 201,55.3 201.4,55.31 201.8,55.33 202.19,55.33 202.59,55.34 202.99,55.33 203.38,55.31 203.78,55.29 204.18,55.29 204.57,55.29 204.97,55.29 205.36,55.28 205.76,55.27 206.16,55.28 206.55,55.29 206.95,55.3 207.35,55.3 207.74,55.29 208.14,55.27 208.54,55.24 208.93,55.22 209.33,55.21 209.73,55.22 210.12,55.25 210.52,55.28 210.92,55.3 211.31,55.3 211.71,55.28 212.11,55.26 212.5,55.24 212.9,55.22 213.3,55.22 213.69,55.21 214.09,55.21 214.49,55.21 214.88,55.2 215.28,55.19 215.68,55.19 216.07,55.2 216.47,55.22 216.86,55.24 217.26,55.25 217.66,55.26 218.05,55.27 218.45,55.27 218.85,55.28 219.24,55.29 219.64,55.31 220.04,55.31 220.43,55.31 220.83,55.29 221.23,55.26 221.62,55.24 222.02,55.22 222.42,55.21 222.81,55.19 223.21,55.19 223.61,55.19 224,55.19 224.4,55.19 224.8,55.19 225.19,55.18 225.59,55.18 225.99,55.19 226.38,55.2 226.78,55.22 227.18,55.22 227.57,55.23 227.97,55.23 228.36,55.24 228.76,55.25 229.16,55.25 229.55,55.26 229.95,55.26 230.35,55.25 230.74,55.24 231.14,55.24 231.54,55.23 231.93,55.23 232.33,55.21 232.73,55.19 233.12,55.17 233.52,55.16 233.92,55.16 234.31,55.15 234.71,55.15 235.11,55.15 235.5,55.15 235.9,55.14 236.3,55.15 236.69,55.16 237.09,55.18 237.49,55.19 237.88,55.19 238.28,55.19 238.68,55.18 239.07,55.17 239.47,55.16 239.86,55.16 240.26,55.15 240.66,55.14 241.05,55.12 241.45,55.11 241.85,55.13 242.24,55.16 242.64,55.21 243.04,55.25 243.43,55.28 243.83,55.29 244.23,55.27 244.62,55.24 245.02,55.22 245.42,55.2 245.81,55.18 246.21,55.17 246.61,55.15 247,55.14 247.4,55.14 247.8,55.15 248.19,55.17 248.59,55.17 248.99,55.18 249.38,55.18 249.78,55.19 250.18,55.19 250.57,55.18 250.97,55.16 251.37,55.13 251.76,55.12 252.16,55.12 252.55,55.13 252.95,55.14 253.35,55.15 253.74,55.16 254.14,55.16 254.54,55.16 254.93,55.16 255.33,55.15 255.73,55.16 256.12,55.16 256.52,55.16 256.92,55.15 257.31,55.14 257.71,55.13 258.11,55.12 258.5,55.12 258.9,55.12 259.3,55.11 259.69,55.09 260.09,55.06 260.49,55.05 260.88,55.04 261.28,55.04 261.68,55.04" stroke-dasharray="none" stroke="rgb(255,255,51)" stroke-width="0.75" opacity="1" stroke-opacity="1" fill="none"/>
          </g>
        </g>
        <g id="Fig:densplot_01.panel.1.1.off.vp.2">
          <g id="Fig:densplot_01.border.panel.1.1.1">
            <rect id="Fig:densplot_01.border.panel.1.1.1.1" x="44.8" y="42.19" width="231.2" height="209.24" transform="" stroke="rgb(0,0,0)" stroke-dasharray="none" stroke-width="0.75" opacity="1" fill="none" stroke-opacity="1" fill-opacity="0"/>
          </g>
        </g>
        <g id="Fig:densplot_01..1"/>
      </g>
    </g>
  </g>
</svg></td><td><table id='Tab:dens'><tr bgcolor='#d0d0ff'><td>array</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>sampleNames</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>title</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>geo_accession</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>status</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>submission_date</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>last_update_date</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>type</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>channel_count</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>source_name_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>organism_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>characteristics_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>characteristics_ch1.1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>characteristics_ch1.2</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>treatment_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>growth_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>molecule_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>extract_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>label_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>label_protocol_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>taxid_ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>hyb_protocol</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>scan_protocol</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>description</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>data_processing</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>platform_id</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_name</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_email</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_laboratory</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_department</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_institute</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_address</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_city</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_state</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>contact_zip/postal_code</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>contact_country</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>supplementary_file</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>data_row_count</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>cell line:ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#e0e0f0'><td>tissue type:ch1</td><td style='font-weight:bold'></td></tr>

<tr bgcolor='#d0d0ff'><td>treatment:ch1</td><td style='font-weight:bold'></td></tr>
</table></td></tr>
</table>
<br>
<b>Figure 6</b><a href="dens.pdf"> (PDF file)</a> shows density estimates (smoothed histograms) of the data. Typically, the distributions of the arrays should have similar shapes and ranges. Arrays whose distributions are very different from the others should be considered for possible problems. Various features of the distributions can be indicative of quality related phenomena. For instance, high levels of background will shift an array's distribution to the right. Lack of signal diminishes its right right tail. A bulge at the upper end of the intensity range often indicates signal saturation.<br><br><br>
</div><hr><h2><a name= 'S3'>Section 3: Variance mean dependence</a></h2>

<a name="msd" id="msd-h" href="javascript: toggle('msd')" style="text-decoration:none;font-weight:bold;font-size:larger">- Figure 7: Standard deviation versus rank of the mean.</a><br>

<div id="msd-b" style="display:block">
<img src="msd.png" border="0" alt="msd.png" id="Figls:msd"></img><br>
<b>Figure 7</b><a href="msd.pdf"> (PDF file)</a> shows a density plot of the standard deviation of the intensities across arrays on the <i>y</i>-axis versus the rank of their mean on the <i>x</i>-axis. The red dots, connected by lines, show the running median of the standard deviation. After normalisation and transformation to a logarithm(-like) scale, one typically expects the red line to be approximately horizontal, that is, show no substantial trend. In some cases, a hump on the right hand of the x-axis can be observed and is symptomatic of a saturation of the intensities.<br><br><br>
</div><hr><h2><a name= 'S4'>Section 4: Individual array quality</a></h2>

<a name="ma" id="ma-h" href="javascript: toggle('ma')" style="text-decoration:none;font-weight:bold;font-size:larger">- Figure 8: MA plots.</a><br>

<div id="ma-b" style="display:block">
<img src="ma.png" border="0" alt="ma.png" id="Figls:ma"></img><br>
<b>Figure 8</b><a href="ma.pdf"> (PDF file)</a> shows MA plots. M and A are defined as:<br> M = log<sub>2</sub>(I<sub>1</sub>) - log<sub>2</sub>(I<sub>2</sub>)<br> A = 1/2 (log<sub>2</sub>(I<sub>1</sub>)+log<sub>2</sub>(I<sub>2</sub>)),<br> where I<sub>1</sub> is the intensity of the array studied,and I<sub>2</sub> is the intensity of a "pseudo"-array that consists of the median across arrays.  Typically, we expect the mass of the distribution in an MA plot to be concentrated along the M = 0 axis,  and there should be no trend in M as a function of A. If there is a trend in the lower range of A, this often  indicates that the arrays have different background intensities; this may be addressed by background correction.  A trend in the upper range of A can indicate saturation of the measurements; in mild cases, this may be addressed  by non-linear normalisation (e.g. quantile normalisation).<br> Outlier detection was performed by computing Hoeffding's statistic <i>D<sub>a</sub></i> on the joint distribution of A and M for each array.   The value of <i>D<sub>a</sub></i> is shown in the panel headings.  0 arrays had <i>D<sub>a</sub></i>&gt;0.15 and were marked as outliers.  For more information on Hoeffing's <i>D</i>-statistic, please see the manual page of the function  <tt>hoeffd</tt> in the <tt>Hmisc</tt> package.<br><br><br>
</div>

<a name="out ma" id="out ma-h" href="javascript: toggle('out ma')" style="text-decoration:none;font-weight:bold;font-size:larger">+ Figure 9: Outlier detection for MA plots.</a><br>

<div id="out ma-b" style="display:none">
<img src="out ma.png" border="0" alt="out ma.png" id="Figls:out ma"></img><br>
<b>Figure 9</b><a href="out ma.pdf"> (PDF file)</a> shows a bar chart of the <i>D<sub>a</sub></i>, the outlier detection criterion from the previous figure. The bars are shown in the original order of the arrays. A threshold of 0.15 was used, which is indicated by the vertical line. None of the arrays exceeded the threshold and was considered an outlier.<br><br><br>
</div><hr><span style="font-size:9pt">This report has been created with arrayQualityMetrics 3.42.0 under R version 3.6.3 (2020-02-29).</span><hr>
<br/><br/><font size="-2">(Page generated on Sat Apr 25 21:30:22 2020 by <a href="http://www.embl.de/~gpau/hwriter/index.html">hwriter</a> )</font><br/>
</body></html>